<?xml version="1.0" encoding="UTF-8"?><xml><records><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Yu, Guoxian</style></author><author><style face="normal" font="default" size="100%">Carlotta Domeniconi</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Guoji Zhang</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Protein Function Prediction using Dependence Maximization</style></title><secondary-title><style face="normal" font="default" size="100%">Proceedings of the European Conference on Machine Learning and Principles of Knowledge Discovery and Data Mining (ECML-PKDD)</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">In Press</style></year><pub-dates><date><style  face="normal" font="default" size="100%">September</style></date></pub-dates></dates><publisher><style face="normal" font="default" size="100%">ECML</style></publisher><pub-location><style face="normal" font="default" size="100%">Prague, CZ</style></pub-location><language><style face="normal" font="default" size="100%">eng</style></language><notes><style face="normal" font="default" size="100%">Acceptance Rate: 111/443 = 25.0%</style></notes></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Sam Blasiak</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Kathryn B. Laskey</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Relevant Subsequence Detection with Sparse Dictionary Learning</style></title><secondary-title><style face="normal" font="default" size="100%">Proceedings of the European Conference on Machine Learning and Principles of Knowledge Discovery and Data Mining (ECML-PKDD)</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">In Press</style></year><pub-dates><date><style  face="normal" font="default" size="100%">September</style></date></pub-dates></dates><publisher><style face="normal" font="default" size="100%">ECML</style></publisher><pub-location><style face="normal" font="default" size="100%">Prague, CZ</style></pub-location><language><style face="normal" font="default" size="100%">eng</style></language><notes><style face="normal" font="default" size="100%">Acceptance Rate: {111/443 = 25.0 %}</style></notes></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Yu, Guoxian</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Carlotta Domeniconi</style></author><author><style face="normal" font="default" size="100%">Guoji Zhang</style></author><author><style face="normal" font="default" size="100%">Zili Zhang</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Protein Function Prediction by Integrating Multiple Kernels</style></title><secondary-title><style face="normal" font="default" size="100%">23rd. International Joint Conference on Artificial Intelligence</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">In Press</style></year><pub-dates><date><style  face="normal" font="default" size="100%">August</style></date></pub-dates></dates><publisher><style face="normal" font="default" size="100%">AAAI</style></publisher><pub-location><style face="normal" font="default" size="100%">Beijing, China</style></pub-location><pages><style face="normal" font="default" size="100%">1‚Äì7</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><notes><style face="normal" font="default" size="100%">Acceptance Rate: {413/1473 = 28%}</style></notes></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Zeehasham Rasheed</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Daniel Barbara</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">16S rRNA Metagenome Clustering and Diversity Estimation using Locality Sensitive Hashing</style></title><secondary-title><style face="normal" font="default" size="100%">BMC Systems Biology</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">In Press</style></year></dates><publisher><style face="normal" font="default" size="100%">BioMed Central</style></publisher><language><style face="normal" font="default" size="100%">eng</style></language><notes><style face="normal" font="default" size="100%">IF: {2.98}</style></notes></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Nuttachat Wisittipanit</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Masoumeh Sikaroodi</style></author><author><style face="normal" font="default" size="100%">ECE Mutlu</style></author><author><style face="normal" font="default" size="100%">Ali Keshavarzian</style></author><author><style face="normal" font="default" size="100%">Patrick Gillevet</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Classification methods for the analysis of LH-PCR data associated with Inflammatory Bowel Disease patients</style></title><secondary-title><style face="normal" font="default" size="100%">International Journal of Bioinformatics Research and Applications</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">In Press</style></year></dates><publisher><style face="normal" font="default" size="100%">Inderdcience</style></publisher><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Siddhartha Sikdar</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Emily Eastlake</style></author><author><style face="normal" font="default" size="100%">Ira Hunt</style></author><author><style face="normal" font="default" size="100%">Andrew Nelson</style></author><author><style face="normal" font="default" size="100%">Jayanth Devanathan</style></author><author><style face="normal" font="default" size="100%">Andrew Shin</style></author><author><style face="normal" font="default" size="100%">Joseph Pancrazio</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Novel Method for Predicting Dexterous Individual Finger Movements by Imaging Muscle Activity using a Wearable Ultrasonic System</style></title><secondary-title><style face="normal" font="default" size="100%">IEEE Transactions on Neural Systems and Rehabilitation Engineering</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">In Press</style></year></dates><publisher><style face="normal" font="default" size="100%">IEEE</style></publisher><language><style face="normal" font="default" size="100%">eng</style></language><notes><style face="normal" font="default" size="100%">IF: {2.4}</style></notes></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Zeehasham Rasheed</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">A Map-Reduce Framework for Clustering Metagenomes</style></title><secondary-title><style face="normal" font="default" size="100%">Proceedings of the 12th IEEE International Workshop on High Performance Computational Biology (HiCOMB)</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2013</style></year><pub-dates><date><style  face="normal" font="default" size="100%">05/2013</style></date></pub-dates></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/HICOMB2013-06.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">IEEE</style></publisher><pub-location><style face="normal" font="default" size="100%">Boston, MA</style></pub-location><pages><style face="normal" font="default" size="100%">549-558</style></pages></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Sam Blasiak</style></author><author><style face="normal" font="default" size="100%">Sithu Sudarshan</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Joint Segmentation and Clustering in Text Corpuses</style></title><secondary-title><style face="normal" font="default" size="100%">SIAM International Conference in Data Mining</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2013</style></year><pub-dates><date><style  face="normal" font="default" size="100%">04/2013</style></date></pub-dates></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/217.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">SIAM</style></publisher><pub-location><style face="normal" font="default" size="100%">Austin, TX</style></pub-location><pages><style face="normal" font="default" size="100%">485-493</style></pages><abstract><style face="normal" font="default" size="100%">In recent years, many private corporations and government organizations have digitized corpuses of legacy paper documents. Often, these organizations hope to take advantage of digital representations to transform costly manual tasks associated with paper archives into less-costly computer-assisted tasks. The most common approach toward automated information extraction is through inverted indexing systems that allow fast keyword searches. Keyword-based indexing, however, is ineffective for tasks that require information from higher level contexts.

To allow for more effective information extraction from digital corpuses, we propose combining two common document processing tasks, (i) clustering and (ii) segmentation, into one process to simultaneously segment documents within a corpus and assigns each segment to a category. We have developed a generative probabilistic model to accomplish this task, which we call the Joint Segmentation and Clustering (JSC) model.

From experiments measuring segmentation and clustering ability, we show that our model can accurately partition documents and assign meaningful categories to each partition. In addition, experiments tracking predictive perplexity show that our JSC model outperforms basic topic modeling approaches in terms of conciseness of the induced representation.</style></abstract><notes><style face="normal" font="default" size="100%">Acceptance Rate: 89/348 = 25.5%.
</style></notes></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Zeehasham Rasheed</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">MC-MinH: Metagenome Clustering using Minwise based Hashing</style></title><secondary-title><style face="normal" font="default" size="100%">SIAM International Conference in Data Mining</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">metagenome clustering</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2013</style></year><pub-dates><date><style  face="normal" font="default" size="100%">04/2013</style></date></pub-dates></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/277.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">SIAM</style></publisher><pub-location><style face="normal" font="default" size="100%">Austin, TX</style></pub-location><pages><style face="normal" font="default" size="100%">677-685</style></pages><abstract><style face="normal" font="default" size="100%">Current bio-technologies allow sequencing of genomes from multiple organisms, that coexist as communities within ecological environments. This collective genomic process (called metagenomics) has spurred the development of several computational tools for the quantification of abundance, diversity and role of different species within different communities. Unsupervised clustering algorithms (also called binning algorithms) have been developed to group similar metagenome sequences.

We have developed an algorithm called MC-MinH that uses the min-wise hashing approach, along with a greedy clustering algorithm to group 16S and whole metagenomic sequences. We represent unequal length sequences using contiguous subsequences or k-mers, and then approximate the computation of pairwise similarity using independent min-wise hashing. The algorithm is written in C and is available using the GNU GPL license at the supplementary website. We evaluate the performance of our algorithm on several real and simulated metagenomic benchmarks. We demonstrate that our approach is computationally efficient and produces accurate clustering results when evaluated using external ground truth.</style></abstract><notes><style face="normal" font="default" size="100%">Acceptance Rate: 89/348 = 25.5%.</style></notes></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Tanwistha Saha</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Carlotta Domeniconi</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Sparsification and Sampling of Networks for Collective Classification</style></title><secondary-title><style face="normal" font="default" size="100%">The 2013 International Conference on Social Computing, Behavioral-Cultural Modeling, &amp; Prediction (SBP)</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2013</style></year><pub-dates><date><style  face="normal" font="default" size="100%">04/2013</style></date></pub-dates></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/10.1007_978-3-642-37210-0_32.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">Springer</style></publisher><pub-location><style face="normal" font="default" size="100%">Washington, DC</style></pub-location><pages><style face="normal" font="default" size="100%">293-302</style></pages><abstract><style face="normal" font="default" size="100%">Network analysis has been an active area of research for the past few decades. Out of many open research questions that have been extensively studied, relational classication, community detection, link prediction are only to name a few. Collective classication is a well-known relational classication method for classifying entities (nodes) within a network which involves using both node based features and topological features of each node. It involves collective prediction of the unknown labels of all the test nodes in the network using label information of the training nodes. Even though this has been a well researched topic for years, very little has been done to address the following two challenges: (1) how to actively select the labeled nodes from the network to be used for training, and (2) how to eciently obtain a sparse representation of the original network without losing much information, so that learning can scale to large networks. A lot of work has been done in theoretical computer science which aims towards nding the best approximation of large graphs. However, not much has been done from the perspective of finding an approximate subgraph that will help in classication of network datasets. In this paper, our contribution is in proposing an efficient graph sparsication method and a sampling technique which, along with the state-of-the-art network classiers, can give comparable runtime and classication accuracies</style></abstract><notes><style face="normal" font="default" size="100%">Acceptance Rate: 24 %</style></notes></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Jasmohan S. Bajaj</style></author><author><style face="normal" font="default" size="100%">Heuman DM</style></author><author><style face="normal" font="default" size="100%">Sanyal AJ</style></author><author><style face="normal" font="default" size="100%">Hylemon PB</style></author><author><style face="normal" font="default" size="100%">Richard K. Sterling,   R. Todd Stravitz,   Michael Fuchs,   Jason M. Ridlon,  Kalyani Daita,   Pamela Monteith,</style></author><author><style face="normal" font="default" size="100%">Nicole A. Noble,   Melanie B. White,   Andmorgan Fisher,   Masoumeh Sikaroodi,</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Patrick Gillevet</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Modulation of the Metabiome by Rifaximin in Patients with Cirrhosis and Minimal Hepatic Encephalopathy</style></title><secondary-title><style face="normal" font="default" size="100%">PLoS ONE </style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2013</style></year></dates><urls><web-urls><url><style face="normal" font="default" size="100%">http://10.1371/journal.pone.0060042</style></url></web-urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/journal.pone_.0060042.pdf</style></url></related-urls></urls><volume><style face="normal" font="default" size="100%">8</style></volume><issue><style face="normal" font="default" size="100%">4</style></issue><call-num><style face="normal" font="default" size="100%">e60042</style></call-num></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Tanwistha Saha</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Carlotta Domeniconi</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Multi-label Classification using Adaptive Neighborhoods</style></title><secondary-title><style face="normal" font="default" size="100%">Eleventh International Conference on Machine Learning and Applications (ICMLA)</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2012</style></year><pub-dates><date><style  face="normal" font="default" size="100%">12/2012</style></date></pub-dates></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/ICMLA_Rank.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">IEEE</style></publisher><pub-location><style face="normal" font="default" size="100%">Boca Raton, FL</style></pub-location><pages><style face="normal" font="default" size="100%">427-432</style></pages></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Anveshi Charuvaka</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Multi-task learning for classifying proteins with dual hierarchies.</style></title><secondary-title><style face="normal" font="default" size="100%">IEEE International Conference on Data Mining (ICDM)</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2012</style></year><pub-dates><date><style  face="normal" font="default" size="100%">12/2012</style></date></pub-dates></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/4905a834.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">IEEE</style></publisher><pub-location><style face="normal" font="default" size="100%">Brussels, Belgium</style></pub-location><pages><style face="normal" font="default" size="100%">834--839</style></pages></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Zeehasham Rasheed</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Daniel Barbara</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">LSH-Div: Species Diversity Estimation using Locality Sensitive Hashing</style></title><secondary-title><style face="normal" font="default" size="100%">IEEE International Conference  on  Bioinformatics and Biomedicine  (BIBM)</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2012</style></year><pub-dates><date><style  face="normal" font="default" size="100%">10/2012</style></date></pub-dates></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/OTU_BIBM2012_Short.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">IEEE</style></publisher><pub-location><style face="normal" font="default" size="100%">Philadelphia, USA</style></pub-location><pages><style face="normal" font="default" size="100%">1 - 6</style></pages><notes><style face="normal" font="default" size="100%">Acceptance Rate: 22.2%</style></notes></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Sam Blasiak</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Kathryn B. Laskey</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">A Family of Feed-forward Models for Protein Sequence Classification</style></title><secondary-title><style face="normal" font="default" size="100%">Proceedings of the European Conference on Machine Learning and Principles of Knowledge Discovery and Data Mining (ECML-PKDD)</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">convolutional neural networks</style></keyword><keyword><style  face="normal" font="default" size="100%">hidden markov models</style></keyword><keyword><style  face="normal" font="default" size="100%">sequence classification</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2012</style></year><pub-dates><date><style  face="normal" font="default" size="100%">09/2012</style></date></pub-dates></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/ecml2012.pdf</style></url></related-urls></urls><number><style face="normal" font="default" size="100%">2</style></number><publisher><style face="normal" font="default" size="100%">ECML</style></publisher><pub-location><style face="normal" font="default" size="100%">Bristol, UK</style></pub-location><pages><style face="normal" font="default" size="100%">419--434</style></pages></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Yu, Guoxian</style></author><author><style face="normal" font="default" size="100%">Carlotta Domeniconi</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Guoji Zhang</style></author><author><style face="normal" font="default" size="100%">Zhiwen Yu</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Transductive multi-label ensemble classification for protein function prediction</style></title><secondary-title><style face="normal" font="default" size="100%">Proceedings of the 18th ACM SIGKDD international conference on Knowledge discovery and data mining</style></secondary-title><tertiary-title><style face="normal" font="default" size="100%">KDD '12</style></tertiary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">directed bi-relation graph</style></keyword><keyword><style  face="normal" font="default" size="100%">multi-label ensemble classifier</style></keyword><keyword><style  face="normal" font="default" size="100%">protein function prediction</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2012</style></year><pub-dates><date><style  face="normal" font="default" size="100%">08/2012</style></date></pub-dates></dates><urls><web-urls><url><style face="normal" font="default" size="100%">http://doi.acm.org/10.1145/2339530.2339700</style></url></web-urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/p1077-yu.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">ACM</style></publisher><pub-location><style face="normal" font="default" size="100%">Beijing, China</style></pub-location><pages><style face="normal" font="default" size="100%">1077‚Äì1085</style></pages><isbn><style face="normal" font="default" size="100%">978-1-4503-1462-6</style></isbn></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Sam Blasiak</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Kathryn B. Laskey</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Beam Methods for the Profile Hidden Markov Model</style></title><secondary-title><style face="normal" font="default" size="100%">2012 SIAM International Conference in Data Mining</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2012</style></year><pub-dates><date><style  face="normal" font="default" size="100%">04/2012</style></date></pub-dates></dates><urls><web-urls><url><style face="normal" font="default" size="100%">http://siam.omnibooksonline.com/2012datamining/</style></url></web-urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/sam2012.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">SIAM</style></publisher><pub-location><style face="normal" font="default" size="100%">Anaheim, CA</style></pub-location><pages><style face="normal" font="default" size="100%">331-342</style></pages></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Zeehasham Rasheed</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Daniel Barbara</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Efficient Clustering of Metagenomic Sequences using Locality Sensitive Hashing</style></title><secondary-title><style face="normal" font="default" size="100%">SIAM International Conference in Data Mining</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">clustering</style></keyword><keyword><style  face="normal" font="default" size="100%">hashing</style></keyword><keyword><style  face="normal" font="default" size="100%">metagenome</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2012</style></year><pub-dates><date><style  face="normal" font="default" size="100%">04/2012</style></date></pub-dates></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/mclsh_sdm2012.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">SIAM</style></publisher><pub-location><style face="normal" font="default" size="100%">Anaheim, CA</style></pub-location><pages><style face="normal" font="default" size="100%">1023-1034</style></pages><abstract><style face="normal" font="default" size="100%">The new generation of genomic technologies have allowed re-
searchers to determine the collective DNA of organisms (e.g.,
microbes) co-existing as communities across the ecosystem
(e.g., within the human host). There is a need for the com-
putational approaches to analyze and annotate the large vol-
umes of available sequence data from such microbial commu-
nities (metagenomes).
In this paper, we developed an ecient and accurate
metagenome clustering approach that uses the locality sen-
sitive hashing (LSH) technique to approximate the computa-
tional complexity associated with comparing sequences. We
introduce the use of xed-length, gapless subsequences for
improving the sensitivity of the LSH-based similarity func-
tion. We evaluate the performance of our algorithm on
two metagenome datasets associated with microbes existing
across dierent human skin locations. Our empirical results
show the strength of the developed approach in compari-
son to three state-of-the-art sequence clustering algorithms
with regards to computational eciency and clustering qual-
ity. We also demonstrate practical signicance for the de-
veloped clustering algorithm, to compare bacterial diversity
and structure across dierent skin locations.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Ece A. Mutlu</style></author><author><style face="normal" font="default" size="100%">Gillevet, Patrick M</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Masoumeh Sikaroodi</style></author><author><style face="normal" font="default" size="100%">Naqvi, Ammar</style></author><author><style face="normal" font="default" size="100%">Engen, Phillip A</style></author><author><style face="normal" font="default" size="100%">Kwasny, Mary</style></author><author><style face="normal" font="default" size="100%">Lau, Cynthia K</style></author><author><style face="normal" font="default" size="100%">Ali Keshavarzian</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Colonic microbiome is altered in alcoholism</style></title><secondary-title><style face="normal" font="default" size="100%">American Journal of Physiology - Gastrointestinal and Liver Physiology</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2012</style></year></dates><urls><web-urls><url><style face="normal" font="default" size="100%">http://ajpgi.physiology.org/content/early/2012/01/06/ajpgi.00380.2011.abstract</style></url></web-urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/Liver Physiol-2012ajpgi.pdf</style></url></related-urls></urls><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">Background: Several studies indicate the importance of colonic microbiota in metabolic and inflammatory disorders and importance of diet on microbiota composition. The effects of alcohol, one of the prominent components of diet, on colonic bacterial composition is largely unknown. Mounting evidence suggests that gut-derived bacterial endotoxins are co-factors for alcohol-induced tissue injury and organ failure like alcoholic liver disease (ALD) that only occur in subset of alcoholics .We hypothesized that chronic alcohol consumption results in alterations of the gut microbiome in a sub-group of alcoholics and this may be responsible for the observed inflammatory state and endotoxemia in alcoholics. Thus, we interrogated the mucosa associated colonic microbiome in 48 alcoholics with and without ALD as well as 18 healthy subjects. Methods: Colonic biopsy samples from subjects were analyzed for microbiota composition using Length Heterogeneity PCR (LH-PCR) fingerprinting and Multitag Pyrosequencing (MTPS). Results: A sub-group of alcoholics have an altered colonic microbiome (dysbiosis). The alcoholics with dysbiosis had lower median abundances of Bacteroidetes and higher ones of Proteobacteria. The observed alterations appear to correlate with high levels of serum endotoxin in a subset of the samples. Network topology analysis indicated that alcohol use is correlated with decreased connectivity of the microbial network and this alteration is seen even after an extended period of sobriety. Conclusion: We show that the colonic mucosa associated bacterial microbiome is altered in a subset of alcoholics. The altered microbiota composition is persistent and correlates with endotoxemia in a sub-group of alcoholics.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Pu Wang</style></author><author><style face="normal" font="default" size="100%">Carlotta Domeniconi</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Kathryn B. Laskey</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Feature Enriched Nonparametric Bayesian Co-clustering</style></title><secondary-title><style face="normal" font="default" size="100%">PAKDD (1)</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2012</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/FE-DPCC.pdf</style></url></related-urls></urls><pages><style face="normal" font="default" size="100%">517-529</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Sharath Hiremagalore</style></author><author><style face="normal" font="default" size="100%">Chen Liang</style></author><author><style face="normal" font="default" size="100%">Angelos Stavrou</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Improving network response times using social information</style></title><secondary-title><style face="normal" font="default" size="100%">Social Network Analysis and Mining</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2012</style></year></dates><urls><web-urls><url><style face="normal" font="default" size="100%">http://dx.doi.org/10.1007/s13278-012-0065-9</style></url></web-urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/SNAM_J_Sharath_2012.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">Springer Wien</style></publisher><pages><style face="normal" font="default" size="100%">1-12</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">Social networks and discussion boards have become significant outlets for people to communicate and freely express their opinions. Although the social networks themselves are usually well-provisioned, the participating users frequently point to external links in order to substantiate their discussions. Unfortunately, the heavy traffic load suddenly imposed on these externally linked websites makes them unresponsive, leading to the ‚Äúflash crowd effect.‚Äù Flash crowds present a real challenge as their intensity and occurrence times are impossible to predict. Moreover, most present-day web hosting servers and caching systems, although increasingly capable, are designed to handle a nominal load of requests before they become unresponsive due to limited bandwidth or the processing power allocated to the hosting site. In this paper, we quantify the prevalence of flash crowd events for a popular social discussion board (Digg). Using PlanetLab, we measured the response times of 1,289 unique popular websites and verified that 89 % of the popular URLs suffered variations in their response times. In an effort to identify flash crowds in advance, we evaluated and compared traffic forecasting mechanisms. We showed that predicting network traffic using network measurements has very limited success and cannot be used for large-scale prediction. However, by analyzing the content and structure of social discussions, we were able to accurately forecast popularity for 86 % of the websites within 5 min of a story‚Äôs submission and for 95% of the sites when more social content (5 h worth) became available. Our work indicates that we can effectively leverage social activity to forecast network events when it would otherwise be infeasible to anticipate them.</style></abstract><notes><style face="normal" font="default" size="100%">10.1007/s13278-012-0065-9</style></notes></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Chaitanya Yavvari</style></author><author><style face="normal" font="default" size="100%">Arnur G. Tokhtabayev</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Angelos Stavrou</style></author></authors><secondary-authors><author><style face="normal" font="default" size="100%">Igor V. Kotenko</style></author><author><style face="normal" font="default" size="100%">Victor A. Skormin</style></author></secondary-authors></contributors><titles><title><style face="normal" font="default" size="100%">Malware Characterization Using Behavioral Components</style></title><secondary-title><style face="normal" font="default" size="100%">MMM-ACNS</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2012</style></year></dates><pub-location><style face="normal" font="default" size="100%">St. Petersburg, Russia</style></pub-location><pages><style face="normal" font="default" size="100%">226-239</style></pages></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Zeehasham Rasheed</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Metagenomic Taxonomic Classification using Extreme Learning Machines. </style></title><secondary-title><style face="normal" font="default" size="100%">Journal of Bioinformatics and Computational Biology</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2012</style></year></dates><urls><web-urls><url><style face="normal" font="default" size="100%">http://www.worldscientific.com/doi/abs/10.1142/S0219720012500151</style></url></web-urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/S0219720012500151.pdf</style></url></related-urls></urls><number><style face="normal" font="default" size="100%">05</style></number><volume><style face="normal" font="default" size="100%">10</style></volume><pages><style face="normal" font="default" size="100%">1250015</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><num-vols><style face="normal" font="default" size="100%">10</style></num-vols></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Yu, G.</style></author><author><style face="normal" font="default" size="100%">Zhang, G.</style></author><author><style face="normal" font="default" size="100%">Rangwala, H.</style></author><author><style face="normal" font="default" size="100%">Domeniconi, C.</style></author><author><style face="normal" font="default" size="100%">Yu, Z.</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Protein function prediction using weak-label learning</style></title><secondary-title><style face="normal" font="default" size="100%">Proceedings of the ACM Conference on Bioinformatics, Computational Biology and Biomedicine</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2012</style></year></dates><publisher><style face="normal" font="default" size="100%">ACM</style></publisher><pages><style face="normal" font="default" size="100%">202‚Äì209</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Anveshi Charuvaka</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Evaluation of short read metagenomic assembly.</style></title><secondary-title><style face="normal" font="default" size="100%">BMC Genomics</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2011</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/1471-2164-12-S2-S8.pdf</style></url></related-urls></urls><pub-location><style face="normal" font="default" size="100%">Computer Science Department, George Mason University, Fairfax, Virginia, USA. rangwala@cs.gmu.edu.</style></pub-location><volume><style face="normal" font="default" size="100%">12 Suppl 2</style></volume><pages><style face="normal" font="default" size="100%">S8</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">BACKGROUND: Metagenomic assembly is a challenging problem due to the presence of genetic material from multiple organisms. The problem becomes even more difficult when short reads produced by next generation sequencing technologies are used. Although whole genome assemblers are not designed to assemble metagenomic samples, they are being used for metagenomics due to the lack of assemblers capable of dealing with metagenomic samples. We present an evaluation of assembly of simulated short-read metagenomic samples using a state-of-art de Bruijn graph based assembler. RESULTS: We assembled simulated metagenomic reads from datasets of various complexities using a state-of-art de Bruijn graph based parallel assembler. We have also studied the effect of k-mer size used in de Bruijn graph on metagenomic assembly and developed a clustering solution to pool the contigs obtained from different assembly runs, which allowed us to obtain longer contigs. We have also assessed the degree of chimericity of the assembled contigs using an entropy/impurity metric and compared the metagenomic assemblies to assemblies of isolated individual source genomes. CONCLUSIONS: Our results show that accuracy of the assembled contigs was better than expected for the metagenomic samples with a few dominant organisms and was especially poor in samples containing many closely related strains. Clustering contigs from different k-mer parameter of the de Bruijn graph allowed us to obtain longer contigs, however the clustering resulted in accumulation of erroneous contigs thus increasing the error rate in clustered contigs.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Tanwistha Saha</style></author><author><style face="normal" font="default" size="100%">Carlotta Domeniconi</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Detection of Communities and Bridges in Weighted Networks</style></title><secondary-title><style face="normal" font="default" size="100%">MLDM</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2011</style></year></dates><pages><style face="normal" font="default" size="100%">584-598</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Syed Faraz Mahmood</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">GPU-Euler: Sequence Assembly Using GPGPU</style></title><secondary-title><style face="normal" font="default" size="100%">HPCC</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2011</style></year></dates><pages><style face="normal" font="default" size="100%">153-160</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Sam Blasiak</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">A Hidden Markov Model Variant for Sequence Classification</style></title><secondary-title><style face="normal" font="default" size="100%">IJCAI</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2011</style></year></dates><pages><style face="normal" font="default" size="100%">1192-1197</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Sheng Li</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">An Information Theoretic Approach for the Analysis of RNA and DNA Binding Sites</style></title><secondary-title><style face="normal" font="default" size="100%">BICoB</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2011</style></year></dates><pages><style face="normal" font="default" size="100%">184-189</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Chen Liang</style></author><author><style face="normal" font="default" size="100%">Sharath Hiremagalore</style></author><author><style face="normal" font="default" size="100%">Angelos Stavrou</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Predicting Network Response Times Using Social Information</style></title><secondary-title><style face="normal" font="default" size="100%">ASONAM</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2011</style></year></dates><pages><style face="normal" font="default" size="100%">527-531</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Nuttachat Wisittipanit</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Masoumeh Sikaroodi</style></author><author><style face="normal" font="default" size="100%">Ali Keshavarzian</style></author><author><style face="normal" font="default" size="100%">Ece A. Mutlu</style></author><author><style face="normal" font="default" size="100%">Patrick Gillevet</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Profiling Microbial Communities using SSU rRNA Sequence and LH-PCR Data</style></title><secondary-title><style face="normal" font="default" size="100%">BICoB</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2011</style></year></dates><pages><style face="normal" font="default" size="100%">190-195</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Dixon, Emma</style></author><author><style face="normal" font="default" size="100%">Clubb, Cynthia</style></author><author><style face="normal" font="default" size="100%">Pittman, Sara</style></author><author><style face="normal" font="default" size="100%">Ammann, Larry</style></author><author><style face="normal" font="default" size="100%">Zeehasham Rasheed</style></author><author><style face="normal" font="default" size="100%">Kazmi, Nazia</style></author><author><style face="normal" font="default" size="100%">Ali Keshavarzian</style></author><author><style face="normal" font="default" size="100%">Gillevet, Pat</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Couch, Robin D</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Solid-phase microextraction and the human fecal VOC metabolome.</style></title><secondary-title><style face="normal" font="default" size="100%">PLoS One</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Metabolome</style></keyword><keyword><style  face="normal" font="default" size="100%">Feces/*chemistry</style></keyword><keyword><style  face="normal" font="default" size="100%">Flame Ionization</style></keyword><keyword><style  face="normal" font="default" size="100%">Gas Chromatography-Mass Spectrometry</style></keyword><keyword><style  face="normal" font="default" size="100%">Hot Temperature</style></keyword><keyword><style  face="normal" font="default" size="100%">Humans</style></keyword><keyword><style  face="normal" font="default" size="100%">Reference Standards</style></keyword><keyword><style  face="normal" font="default" size="100%">Solid Phase Microextraction/*methods</style></keyword><keyword><style  face="normal" font="default" size="100%">Sterilization</style></keyword><keyword><style  face="normal" font="default" size="100%">Time Factors</style></keyword><keyword><style  face="normal" font="default" size="100%">Volatile Organic Compounds/*analysis</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2011</style></year></dates><number><style face="normal" font="default" size="100%">4</style></number><pub-location><style face="normal" font="default" size="100%">Department of Chemistry and Biochemistry, George Mason University, Manassas, Virginia, United States of America.</style></pub-location><volume><style face="normal" font="default" size="100%">6</style></volume><pages><style face="normal" font="default" size="100%">e18471</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">The diagnostic potential and health implications of volatile organic compounds (VOCs) present in human feces has begun to receive considerable attention. Headspace solid-phase microextraction (SPME) has greatly facilitated the isolation and analysis of VOCs from human feces. Pioneering human fecal VOC metabolomic investigations have utilized a single SPME fiber type for analyte extraction and analysis. However, we hypothesized that the multifarious nature of metabolites present in human feces dictates the use of several diverse SPME fiber coatings for more comprehensive metabolomic coverage. We report here an evaluation of eight different commercially available SPME fibers, in combination with both GC-MS and GC-FID, and identify the 50/30 microm CAR-DVB-PDMS, 85 microm CAR-PDMS, 65 microm DVB-PDMS, 7 microm PDMS, and 60 microm PEG SPME fibers as a minimal set of fibers appropriate for human fecal VOC metabolomics, collectively isolating approximately 90% of the total metabolites obtained when using all eight fibers. We also evaluate the effect of extraction duration on metabolite isolation and illustrate that ex vivo enteric microbial fermentation has no effect on metabolite composition during prolonged extractions if the SPME is performed as described herein.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Zeehasham Rasheed</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">TAC-ELM: Metagenomic Taxonomic Classification with Extreme Learning Machines</style></title><secondary-title><style face="normal" font="default" size="100%">BICoB</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2011</style></year></dates><pages><style face="normal" font="default" size="100%">92-97</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Salman Jamali</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Defining a Coparticipation Network Using Comments on Digg</style></title><secondary-title><style face="normal" font="default" size="100%">IEEE Intelligent Systems</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/05552588.pdf</style></url></related-urls></urls><number><style face="normal" font="default" size="100%">4</style></number><volume><style face="normal" font="default" size="100%">25</style></volume><pages><style face="normal" font="default" size="100%">36-45</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Charaniya, Salim</style></author><author><style face="normal" font="default" size="100%">Le, Huong</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Mills, Keri</style></author><author><style face="normal" font="default" size="100%">Johnson, Kevin</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author><author><style face="normal" font="default" size="100%">Hu, Wei-Shou</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Mining manufacturing data for discovery of high productivity process characteristics</style></title><secondary-title><style face="normal" font="default" size="100%">Journal of biotechnology</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/1-s2.0-S0168165610001902-main.pdf</style></url></related-urls></urls><number><style face="normal" font="default" size="100%">3</style></number><publisher><style face="normal" font="default" size="100%">Elsevier</style></publisher><volume><style face="normal" font="default" size="100%">147</style></volume><pages><style face="normal" font="default" size="100%">186‚Äì197</style></pages></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Naqvi, Ammar</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Spear, Greg</style></author><author><style face="normal" font="default" size="100%">Patrick Gillevet</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Analysis of multitag pyrosequence data from human cervical lavage samples.</style></title><secondary-title><style face="normal" font="default" size="100%">Chem Biodivers</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Metagenome</style></keyword><keyword><style  face="normal" font="default" size="100%">*Sequence Analysis</style></keyword><keyword><style  face="normal" font="default" size="100%">16S/genetics</style></keyword><keyword><style  face="normal" font="default" size="100%">Bacteria/*classification/genetics/isolation &amp; purification</style></keyword><keyword><style  face="normal" font="default" size="100%">Bacterial/complications/*microbiology</style></keyword><keyword><style  face="normal" font="default" size="100%">Cervix Uteri</style></keyword><keyword><style  face="normal" font="default" size="100%">Cluster Analysis</style></keyword><keyword><style  face="normal" font="default" size="100%">DNA</style></keyword><keyword><style  face="normal" font="default" size="100%">Female</style></keyword><keyword><style  face="normal" font="default" size="100%">HIV Infections/complications</style></keyword><keyword><style  face="normal" font="default" size="100%">Humans</style></keyword><keyword><style  face="normal" font="default" size="100%">Ribosomal</style></keyword><keyword><style  face="normal" font="default" size="100%">RNA</style></keyword><keyword><style  face="normal" font="default" size="100%">Therapeutic Irrigation</style></keyword><keyword><style  face="normal" font="default" size="100%">Vaginosis</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><number><style face="normal" font="default" size="100%">5</style></number><pub-location><style face="normal" font="default" size="100%">Microbiome Analysis Center, Manassas, VA 20110, USA.</style></pub-location><volume><style face="normal" font="default" size="100%">7</style></volume><pages><style face="normal" font="default" size="100%">1076‚Äì1085</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">We have been using the Roche GS-FLX sequencing platform to produce tens of thousands of sequencing reads from samples of both bacterial communities (microbiome) and fungal communities (mycobiome) of stool, gut mucosa, vaginal washes, and oral washes from a large number of subjects. This vast volume of data from diverse sources has necessitated the development of an analysis pipeline in order to systematically and rapidly identify the taxa within the samples and to correlate the sample data with clinical and environmental features. Specifically, we have developed automated analytical tools for data tracking, taxonomical analysis, and feature clustering of bacteria in the human microbiome and demonstrate the pipeline using Cervical Vaginal Lavage (CVL) samples. This analysis pipeline will not only provide insight to our specific CVL dataset, but is applicable to other microbiome samples and will ultimately broaden our understanding of how the microbiome influences human health.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Salman Jamali</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Co-Participation Networks Using Comment Information</style></title><secondary-title><style face="normal" font="default" size="100%">ICWSM</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Anveshi Charuvaka</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Evaluation of short read metagenomic assembly</style></title><secondary-title><style face="normal" font="default" size="100%">BIBM</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><pages><style face="normal" font="default" size="100%">171-178</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>6</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Karypis, G</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Introduction to protein structure prediction : methods and algorithms</style></title><tertiary-title><style face="normal" font="default" size="100%">Wiley series on bioinformatics</style></tertiary-title></titles><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><urls><web-urls><url><style face="normal" font="default" size="100%">http://www.loc.gov/catdir/enhancements/fy1112/2010028352-b.html</style></url></web-urls></urls><publisher><style face="normal" font="default" size="100%">Wiley</style></publisher><pub-location><style face="normal" font="default" size="100%">Hoboken, N.J.</style></pub-location><volume><style face="normal" font="default" size="100%">14</style></volume><isbn><style face="normal" font="default" size="100%">9780470470596</style></isbn><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">&quot;This book helps unravel the relationship of pure sequence information and three-dimensional structure, which remains one of the great fundamental problems in molecular biology and bioinformatics. It describes key applications of modeled structures, focusing on the methods and algorithms that are used to predict protein structure written by experts who participate in the structure prediction competition. The book also delivers applications used for predicted models in other studies. Researchers in bioinformatics and molecular biology will find this text highly useful, as will students in graduate courses in protein prediction&quot;‚Äì</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>5</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors><secondary-authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></secondary-authors></contributors><titles><title><style face="normal" font="default" size="100%">Introduction to Protein Structure Prediction: Methods and Algorithms</style></title></titles><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><publisher><style face="normal" font="default" size="100%">Wiley</style></publisher><pages><style face="normal" font="default" size="100%">1‚Äì14</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><section><style face="normal" font="default" size="100%">Introduction to Protein Structure</style></section></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>5</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors><secondary-authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></secondary-authors></contributors><titles><title><style face="normal" font="default" size="100%">Introduction to Protein Structure Prediction: Methods and Algorithms</style></title></titles><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><publisher><style face="normal" font="default" size="100%">Wiley</style></publisher><pages><style face="normal" font="default" size="100%">165‚Äì194</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><section><style face="normal" font="default" size="100%">A survey of remote homology detection methods</style></section></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Charaniya, Salim</style></author><author><style face="normal" font="default" size="100%">Le, Huong</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Mills, Keri</style></author><author><style face="normal" font="default" size="100%">Johnson, Kevin</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author><author><style face="normal" font="default" size="100%">Hu, Wei-Shou</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Mining manufacturing data for discovery of high productivity process characteristics.</style></title><secondary-title><style face="normal" font="default" size="100%">J Biotechnol</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Bioreactors</style></keyword><keyword><style  face="normal" font="default" size="100%">Algorithms</style></keyword><keyword><style  face="normal" font="default" size="100%">Biotechnology/*methods</style></keyword><keyword><style  face="normal" font="default" size="100%">Cluster Analysis</style></keyword><keyword><style  face="normal" font="default" size="100%">Data Mining/*methods</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><number><style face="normal" font="default" size="100%">3-4</style></number><pub-location><style face="normal" font="default" size="100%">Department of Chemical Engineering and Materials Science, University of Minnesota, 421 Washington Avenue SE, Minneapolis, MN 55455-0132, USA.</style></pub-location><volume><style face="normal" font="default" size="100%">147</style></volume><pages><style face="normal" font="default" size="100%">186‚Äì197</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">Modern manufacturing facilities for bioproducts are highly automated with advanced process monitoring and data archiving systems. The time dynamics of hundreds of process parameters and outcome variables over a large number of production runs are archived in the data warehouse. This vast amount of data is a vital resource to comprehend the complex characteristics of bioprocesses and enhance production robustness. Cell culture process data from 108 'trains' comprising production as well as inoculum bioreactors from Genentech's manufacturing facility were investigated. Each run constitutes over one-hundred on-line and off-line temporal parameters. A kernel-based approach combined with a maximum margin-based support vector regression algorithm was used to integrate all the process parameters and develop predictive models for a key cell culture performance parameter. The model was also used to identify and rank process parameters according to their relevance in predicting process outcome. Evaluation of cell culture stage-specific models indicates that production performance can be reliably predicted days prior to harvest. Strong associations between several temporal parameters at various manufacturing stages and final process outcome were uncovered. This model-based data mining represents an important step forward in establishing a process data-driven knowledge discovery in bioprocesses. Implementation of this methodology on the manufacturing floor can facilitate a real-time decision making process and thereby improve the robustness of large scale bioprocesses.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Multiple Kernel Learning for Fold Recognition</style></title><secondary-title><style face="normal" font="default" size="100%">BICoB</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><pages><style face="normal" font="default" size="100%">7-12</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Naqvi, Ammar</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Ali Keshavarzian</style></author><author><style face="normal" font="default" size="100%">Patrick Gillevet</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Network-based modeling of the human gut microbiome.</style></title><secondary-title><style face="normal" font="default" size="100%">Chem Biodivers</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Metagenome</style></keyword><keyword><style  face="normal" font="default" size="100%">*Models</style></keyword><keyword><style  face="normal" font="default" size="100%">Biological</style></keyword><keyword><style  face="normal" font="default" size="100%">Colon/*microbiology</style></keyword><keyword><style  face="normal" font="default" size="100%">Humans</style></keyword><keyword><style  face="normal" font="default" size="100%">Metabolic Networks and Pathways</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><number><style face="normal" font="default" size="100%">5</style></number><pub-location><style face="normal" font="default" size="100%">Microbiome Analysis Center, Manassas, VA 20110, USA.</style></pub-location><volume><style face="normal" font="default" size="100%">7</style></volume><pages><style face="normal" font="default" size="100%">1040‚Äì1050</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">In this article, we used a network-based approach to characterize the microflora abundance in colonic mucosal samples and correlate potential interactions between the identified species with respect to the healthy and diseased states. We analyzed the modelled network by computing several local and global network statistics, identified recurring patterns or motifs, fit the network models to a family of well-studied graph models. This study has demonstrated, for the first time, an approach that differentiated the gut microbiota in alcoholic subjects and healthy subjects using topological network analysis of the gut microbiome.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Nuttachat Wisittipanit</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Patrick Gillevet</style></author><author><style face="normal" font="default" size="100%">Masoumeh Sikaroodi</style></author><author><style face="normal" font="default" size="100%">Ece A. Mutlu</style></author><author><style face="normal" font="default" size="100%">Ali Keshavarzian</style></author></authors><secondary-authors><author><style face="normal" font="default" size="100%">Jun Huan</style></author><author><style face="normal" font="default" size="100%">Jake Chen</style></author><author><style face="normal" font="default" size="100%">Mohammed Zaki</style></author></secondary-authors></contributors><titles><title><style face="normal" font="default" size="100%">SVM-based classification and feature selection methods for the analysis of Inflammatory Bowel disease microbiome data</style></title><secondary-title><style face="normal" font="default" size="100%">Proceedings of the 9th International Workshop on Data Mining in Bioinformatics</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><pages><style face="normal" font="default" size="100%">1-8</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Rezwan Ahmed</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">TOPTMH: topology predictor for transmembrane alpha-helices.</style></title><secondary-title><style face="normal" font="default" size="100%">J Bioinform Comput Biol</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Algorithms</style></keyword><keyword><style  face="normal" font="default" size="100%">*Models</style></keyword><keyword><style  face="normal" font="default" size="100%">*Protein Structure</style></keyword><keyword><style  face="normal" font="default" size="100%">Artificial Intelligence</style></keyword><keyword><style  face="normal" font="default" size="100%">Computational Biology</style></keyword><keyword><style  face="normal" font="default" size="100%">Databases</style></keyword><keyword><style  face="normal" font="default" size="100%">Hydrophobic and Hydrophilic Interactions</style></keyword><keyword><style  face="normal" font="default" size="100%">Markov Chains</style></keyword><keyword><style  face="normal" font="default" size="100%">Membrane Proteins/*chemistry</style></keyword><keyword><style  face="normal" font="default" size="100%">Molecular</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein</style></keyword><keyword><style  face="normal" font="default" size="100%">Secondary</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2010</style></year></dates><number><style face="normal" font="default" size="100%">1</style></number><pub-location><style face="normal" font="default" size="100%">Department of Computer Science and Engineering, University of Minnesota, Minneapolis, Minnesota 55455, USA. ahmed@cs.umn.edu</style></pub-location><volume><style face="normal" font="default" size="100%">8</style></volume><pages><style face="normal" font="default" size="100%">39‚Äì57</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">Alpha-helical transmembrane proteins mediate many key biological processes and represent 20%-30% of all genes in many organisms. Due to the difficulties in experimentally determining their high-resolution 3D structure, computational methods to predict the location and orientation of transmembrane helix segments using sequence information are essential. We present TOPTMH, a new transmembrane helix topology prediction method that combines support vector machines, hidden Markov models, and a widely used rule-based scheme. The contribution of this work is the development of a prediction approach that first uses a binary SVM classifier to predict the helix residues and then it employs a pair of HMM models that incorporate the SVM predictions and hydropathy-based features to identify the entire transmembrane helix segments by capturing the structural characteristics of these proteins. TOPTMH outperforms state-of-the-art prediction methods and achieves the best performance on an independent static benchmark.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Salman Jamali</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Digging Digg: Comment Mining, Popularity Prediction, and Social Network Analysis</style></title><secondary-title><style face="normal" font="default" size="100%">Web Information Systems and Mining, International Conference on</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2009</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/GMU-CS-TR-2009-7.pdf</style></url></related-urls></urls><publisher><style face="normal" font="default" size="100%">IEEE Computer Society</style></publisher><pub-location><style face="normal" font="default" size="100%">Los Alamitos, CA, USA</style></pub-location><pages><style face="normal" font="default" size="100%">32-38</style></pages><isbn><style face="normal" font="default" size="100%">978-0-7695-3817-4</style></isbn><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Ning, Xia</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Multi-assay-based structure-activity relationship models: improving structure-activity relationship models by incorporating activity information from related targets.</style></title><secondary-title><style face="normal" font="default" size="100%">J Chem Inf Model</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Models</style></keyword><keyword><style  face="normal" font="default" size="100%">Chemical</style></keyword><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword><keyword><style  face="normal" font="default" size="100%">Structure-Activity Relationship</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2009</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/Ning2009.pdf</style></url></related-urls></urls><number><style face="normal" font="default" size="100%">11</style></number><pub-location><style face="normal" font="default" size="100%">Department of Computer Science and Computer Engineering, University of Minnesota, 4-192 EE/CS Building, 200 Union Street SE, Minneapolis, Minnesota 55455, USA. xning@cs.umn.edu</style></pub-location><volume><style face="normal" font="default" size="100%">49</style></volume><pages><style face="normal" font="default" size="100%">2444‚Äì2456</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">Structure-activity relationship (SAR) models are used to inform and to guide the iterative optimization of chemical leads, and they play a fundamental role in modern drug discovery. In this paper, we present a new class of methods for building SAR models, referred to as multi-assay based, that utilize activity information from different targets. These methods first identify a set of targets that are related to the target under consideration, and then they employ various machine learning techniques that utilize activity information from these targets in order to build the desired SAR model. We developed different methods for identifying the set of related targets, which take into account the primary sequence of the targets or the structure of their ligands, and we also developed different machine learning techniques that were derived by using principles of semi-supervised learning, multi-task learning, and classifier ensembles. The comprehensive evaluation of these methods shows that they lead to considerable improvements over the standard SAR models that are based only on the ligands of the target under consideration. On a set of 117 protein targets, obtained from PubChem, these multi-assay-based methods achieve a receiver-operating characteristic score that is, on the average, 7.0 -7.2% higher than that achieved by the standard SAR models. Moreover, on a set of targets belonging to six protein families, the multi-assay-based methods outperform chemogenomics-based approaches by 4.33%.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Christopher Kauffman</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">svmPRAT: SVM-based protein residue annotation toolkit.</style></title><secondary-title><style face="normal" font="default" size="100%">BMC Bioinformatics</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Software</style></keyword><keyword><style  face="normal" font="default" size="100%">Artificial Intelligence</style></keyword><keyword><style  face="normal" font="default" size="100%">Automated</style></keyword><keyword><style  face="normal" font="default" size="100%">Binding Sites</style></keyword><keyword><style  face="normal" font="default" size="100%">Databases</style></keyword><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword><keyword><style  face="normal" font="default" size="100%">Pattern Recognition</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein Folding</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein Structure</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein/*methods</style></keyword><keyword><style  face="normal" font="default" size="100%">Proteins/*chemistry</style></keyword><keyword><style  face="normal" font="default" size="100%">Secondary</style></keyword><keyword><style  face="normal" font="default" size="100%">Sequence Analysis</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2009</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/1471-2105-10-439_0.pdf</style></url></related-urls></urls><pub-location><style face="normal" font="default" size="100%">Computer Science Department, George Mason University, Fairfax, VA, USA. rangwala@cs.gmu.edu</style></pub-location><volume><style face="normal" font="default" size="100%">10</style></volume><pages><style face="normal" font="default" size="100%">439</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">BACKGROUND: Over the last decade several prediction methods have been developed for determining the structural and functional properties of individual protein residues using sequence and sequence-derived information. Most of these methods are based on support vector machines as they provide accurate and generalizable prediction models. RESULTS: We present a general purpose protein residue annotation toolkit (svmPRAT) to allow biologists to formulate residue-wise prediction problems. svmPRAT formulates the annotation problem as a classification or regression problem using support vector machines. One of the key features of svmPRAT is its ease of use in incorporating any user-provided information in the form of feature matrices. For every residue svmPRAT captures local information around the reside to create fixed length feature vectors. svmPRAT implements accurate and fast kernel functions, and also introduces a flexible window-based encoding scheme that accurately captures signals and pattern for training effective predictive models. CONCLUSIONS: In this work we evaluate svmPRAT on several classification and regression problems including disorder prediction, residue-wise contact order estimation, DNA-binding site prediction, and local structure alphabet prediction. svmPRAT has also been used for the development of state-of-the-art transmembrane helix prediction method called TOPTMH, and secondary structure prediction method called YASSPP. This toolkit developed provides practitioners an efficient and easy-to-use tool for a wide variety of annotation problems.Availability: http://www.cs.gmu.edu/¬†mlbio/svmprat.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>5</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors><secondary-authors><author><style face="normal" font="default" size="100%">Jake Chen</style></author><author><style face="normal" font="default" size="100%">Stefano Lonardi</style></author></secondary-authors></contributors><titles><title><style face="normal" font="default" size="100%">Biological Data Mining</style></title></titles><dates><year><style  face="normal" font="default" size="100%">2009</style></year></dates><publisher><style face="normal" font="default" size="100%">CRC Press</style></publisher><language><style face="normal" font="default" size="100%">eng</style></language><section><style face="normal" font="default" size="100%">Predicting Local Structure and Function of Proteins</style></section></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Christopher Kauffman</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">A Kernel Framework for Protein Residue Annotation</style></title><secondary-title><style face="normal" font="default" size="100%">PAKDD</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2009</style></year></dates><pages><style face="normal" font="default" size="100%">439-451</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">Over the last decade several prediction methods have been developed for determining structural and functional properties of individual protein residues using sequence and sequence-derived information. Most of these methods are based on support vector machines as they provide accurate and generalizable prediction models. We developed a general purpose protein residue annotation toolkit (Pro SAT) to allow biologists to formulate residue-wise prediction problems. Pro SAT formulates annotation problem as a classification or regression problem using support vector machines. For every residue Pro SAT captures local information (any sequence-derived information) around the reside to create fixed length feature vectors. Pro SAT implements accurate and fast kernel functions, and also introduces a flexible window-based encoding scheme that allows better capture of signals for certain prediction problems. In this work we evaluate the performance of Pro SAT on the disorder prediction and contact order estimation problems, studying the effect of the different kernels introduced here. Pro SAT shows better or at least comparable performance to state-of-the-art prediction systems. In particular Pro SAT has proven to be the best performing transmembrane-helix predictor on an independent blind benchmark. Availability: http://bio.dtc.umn.edu/prosat</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">fRMSDPred: predicting local RMSD between structural fragments using sequence information.</style></title><secondary-title><style face="normal" font="default" size="100%">Proteins</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Algorithms</style></keyword><keyword><style  face="normal" font="default" size="100%">*Sequence Analysis</style></keyword><keyword><style  face="normal" font="default" size="100%">Databases</style></keyword><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein</style></keyword><keyword><style  face="normal" font="default" size="100%">Proteins</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2008</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/21998_ftp.pdf</style></url></related-urls></urls><number><style face="normal" font="default" size="100%">3</style></number><pub-location><style face="normal" font="default" size="100%">Computer Science and Engineering, University of Minnesota, Minneapolis, Minnesota 55455, USA. rangwala@cs.umn.edu</style></pub-location><volume><style face="normal" font="default" size="100%">72</style></volume><pages><style face="normal" font="default" size="100%">1005‚Äì1018</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">The effectiveness of comparative modeling approaches for protein structure prediction can be substantially improved by incorporating predicted structural information in the initial sequence-structure alignment. Motivated by the approaches used to align protein structures, this article focuses on developing machine learning approaches for estimating the RMSD value of a pair of protein fragments. These estimated fragment-level RMSD values can be used to construct the alignment, assess the quality of an alignment, and identify high-quality alignment segments. We present algorithms to solve this fragment-level RMSD prediction problem using a supervised learning framework based on support vector regression and classification that incorporates protein profiles, predicted secondary structure, effective information encoding schemes, and novel second-order pairwise exponential kernel functions. Our comprehensive empirical study shows superior results compared with the profile-to-profile scoring schemes. We also show that for protein pairs with low sequence similarity (less than 12% sequence identity) these new local structural features alone or in conjunction with profile-based information lead to alignments that are considerably accurate than those obtained by schemes that use only profile and/or predicted secondary structure information.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">fRMSDAlign: Protein Sequence Alignment Using Predicted Local Structure Information for Pairs with Low Sequence Identity</style></title><secondary-title><style face="normal" font="default" size="100%">APBC</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2008</style></year></dates><pages><style face="normal" font="default" size="100%">111-122</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Ruinan Zhang</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Genome Alignments Using MPI-LAGAN</style></title><secondary-title><style face="normal" font="default" size="100%">BIBM</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2008</style></year></dates><pages><style face="normal" font="default" size="100%">437-440</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Kauffman, Chris</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Improving homology models for protein-ligand binding sites.</style></title><secondary-title><style face="normal" font="default" size="100%">Comput Syst Bioinformatics Conf</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Algorithms</style></keyword><keyword><style  face="normal" font="default" size="100%">*Models</style></keyword><keyword><style  face="normal" font="default" size="100%">Amino Acid</style></keyword><keyword><style  face="normal" font="default" size="100%">Amino Acid Sequence</style></keyword><keyword><style  face="normal" font="default" size="100%">Binding Sites</style></keyword><keyword><style  face="normal" font="default" size="100%">Chemical</style></keyword><keyword><style  face="normal" font="default" size="100%">Computer Simulation</style></keyword><keyword><style  face="normal" font="default" size="100%">Ligands</style></keyword><keyword><style  face="normal" font="default" size="100%">Molecular</style></keyword><keyword><style  face="normal" font="default" size="100%">Molecular Sequence Data</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein Binding</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein Conformation</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein Structure</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein/*methods</style></keyword><keyword><style  face="normal" font="default" size="100%">Proteins/*chemistry/*ultrastructure</style></keyword><keyword><style  face="normal" font="default" size="100%">Sequence Alignment/*methods</style></keyword><keyword><style  face="normal" font="default" size="100%">Sequence Analysis</style></keyword><keyword><style  face="normal" font="default" size="100%">Sequence Homology</style></keyword><keyword><style  face="normal" font="default" size="100%">Tertiary</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2008</style></year></dates><pub-location><style face="normal" font="default" size="100%">Department of Computer Science, University of Minnesota, 117 Pleasant St SE, Room 464, Minneapolis, MN 55455, USA. kauffman@cs.umn.edu</style></pub-location><volume><style face="normal" font="default" size="100%">7</style></volume><pages><style face="normal" font="default" size="100%">211‚Äì222</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">In order to improve the prediction of protein-ligand binding sites through homology modeling, we incorporate knowledge of the binding residues into the modeling framework. Residues are identified as binding or nonbinding based on their true labels as well as labels predicted from structure and sequence. The sequence predictions were made using a support vector machine framework which employs a sophisticated window-based kernel. Binding labels are used with a very sensitive sequence alignment method to align the target and template. Relevant parameters governing the alignment process are searched for optimal values. Based on our results, homology models of the binding site can be improved if a priori knowledge of the binding residues is available. For target-template pairs with low sequence identity and high structural diversity our sequence-based prediction method provided sufficient information to realize this improvement.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Rezwan Ahmed</style></author><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">TOPTMH: Topology Predictor for Transmembrane alpha-Helices</style></title><secondary-title><style face="normal" font="default" size="100%">ECML/PKDD (1)</style></secondary-title></titles><dates><year><style  face="normal" font="default" size="100%">2008</style></year></dates><pages><style face="normal" font="default" size="100%">23-38</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">Alpha-helical transmembrane proteins mediate many key biological processes and represent 20%¬ñ30% of all genes in many organisms. Due to the difficulties in experimentally determining their high-resolution 3D structure, computational methods to predict the location and orientation of transmembrane helix segments using sequence information are essential. We present, TOPTMH a new transmembrane helix topology prediction method that combines support vector machines, hidden Markov models, and a widely-used rule-based scheme. The contribution of this work is the development of a prediction approach that first uses a binary SVM classifier to predict the helix residues and then it employs a pair of HMM models that incorporate the SVM predictions and hydropathy-based features to identify the entire transmembrane helix segments by capturing the structural characteristics of these proteins. TOPTMH outperforms state-of-the-art prediction methods and achieves the best performance on an independent static benchmark.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Incremental window-based protein sequence alignment algorithms.</style></title><secondary-title><style face="normal" font="default" size="100%">Bioinformatics</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Algorithms</style></keyword><keyword><style  face="normal" font="default" size="100%">Amino Acid Sequence</style></keyword><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword><keyword><style  face="normal" font="default" size="100%">Molecular Sequence Data</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein/*methods</style></keyword><keyword><style  face="normal" font="default" size="100%">Proteins/*chemistry</style></keyword><keyword><style  face="normal" font="default" size="100%">Sequence Alignment/*methods</style></keyword><keyword><style  face="normal" font="default" size="100%">Sequence Analysis</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2007</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/Bioinformatics-2007-Rangwala-e17-23.pdf</style></url></related-urls></urls><number><style face="normal" font="default" size="100%">2</style></number><pub-location><style face="normal" font="default" size="100%">Department of Computer Science &amp; Engineering, University of Minnesota Minneapolis, MN 55455, USA. rangwala@cs.umn.edu</style></pub-location><volume><style face="normal" font="default" size="100%">23</style></volume><pages><style face="normal" font="default" size="100%">e17-23</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">MOTIVATION: Protein sequence alignment plays a critical role in computational biology as it is an integral part in many analysis tasks designed to solve problems in comparative genomics, structure and function prediction, and homology modeling. METHODS: We have developed novel sequence alignment algorithms that compute the alignment between a pair of sequences based on short fixed- or variable-length high-scoring subsequences. Our algorithms build the alignments by repeatedly selecting the highest scoring pairs of subsequences and using them to construct small portions of the final alignment. We utilize PSI-BLAST generated sequence profiles and employ a profile-to-profile scoring scheme derived from PICASSO. RESULTS: We evaluated the performance of the computed alignments on two recently published benchmark datasets and compared them against the alignments computed by existing state-of-the-art dynamic programming-based profile-to-profile local and global sequence alignment algorithms. Our results show that the new algorithms achieve alignments that are comparable with or better than those achieved by existing algorithms. Moreover, our results also showed that these algorithms can be used to provide better information as to which of the aligned positions are more reliable‚Äìa critical piece of information for comparative modeling applications.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">fRMSDPred: predicting local RMSD between structural fragments using sequence information.</style></title><secondary-title><style face="normal" font="default" size="100%">Comput Syst Bioinformatics Conf</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Models</style></keyword><keyword><style  face="normal" font="default" size="100%">Chemical</style></keyword><keyword><style  face="normal" font="default" size="100%">Computer Simulation</style></keyword><keyword><style  face="normal" font="default" size="100%">Least-Squares Analysis</style></keyword><keyword><style  face="normal" font="default" size="100%">Models</style></keyword><keyword><style  face="normal" font="default" size="100%">Molecular</style></keyword><keyword><style  face="normal" font="default" size="100%">Peptide Fragments/*chemistry</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein Conformation</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein Folding</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein/*methods</style></keyword><keyword><style  face="normal" font="default" size="100%">Proteins/*chemistry/*ultrastructure</style></keyword><keyword><style  face="normal" font="default" size="100%">Sequence Analysis</style></keyword><keyword><style  face="normal" font="default" size="100%">Statistical</style></keyword><keyword><style  face="normal" font="default" size="100%">Surface Properties</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2007</style></year></dates><pub-location><style face="normal" font="default" size="100%">Computer Science &amp; Engineering, University of Minnesota, Minneapolis, MN 55455, USA. rangwala@cs.umn.edu</style></pub-location><volume><style face="normal" font="default" size="100%">6</style></volume><pages><style face="normal" font="default" size="100%">311‚Äì322</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">The effectiveness of comparative modeling approaches for protein structure prediction can be substantially improved by incorporating predicted structural information in the initial sequence-structure alignment. Motivated by the approaches used to align protein structures, this paper focuses on developing machine learning approaches for estimating the RMSD value of a pair of protein fragments. These estimated fragment-level RMSD values can be used to construct the alignment, assess the quality of an alignment, and identify high-quality alignment segments. We present algorithms to solve this fragment-level RMSD prediction problem using a supervised learning framework based on support vector regression and classification that incorporates protein profiles, predicted secondary structure, effective information encoding schemes, and novel second-order pairwise exponential kernel functions. Our comprehensive empirical study shows superior results compared to the profile-to-profile scoring schemes.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>5</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Kevin DeRonne</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors><secondary-authors><author><style face="normal" font="default" size="100%">Xiaohua Hu</style></author><author><style face="normal" font="default" size="100%">Yi Pan</style></author></secondary-authors></contributors><titles><title><style face="normal" font="default" size="100%">Knowledge Discovery in Bioinformatics: Techniques, Methods, and Applications</style></title><secondary-title><style face="normal" font="default" size="100%">Knowledge Discovery in Bioinformatics</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">protein structure prediction using string kernels</style></keyword><keyword><style  face="normal" font="default" size="100%">remote homology and fold prediction</style></keyword><keyword><style  face="normal" font="default" size="100%">secondary-structure prediction</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2007</style></year></dates><urls><web-urls><url><style face="normal" font="default" size="100%">http://dx.doi.org/10.1002/9780470124642.ch8</style></url></web-urls></urls><publisher><style face="normal" font="default" size="100%">John Wiley &amp; Sons, Inc.</style></publisher><pages><style face="normal" font="default" size="100%">145‚Äì168</style></pages><isbn><style face="normal" font="default" size="100%">9780470124642</style></isbn><language><style face="normal" font="default" size="100%">eng</style></language><section><style face="normal" font="default" size="100%">Protein Structure Prediction using String Kernels</style></section></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Building multiclass classifiers for remote homology detection and fold recognition.</style></title><secondary-title><style face="normal" font="default" size="100%">BMC Bioinformatics</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Algorithms</style></keyword><keyword><style  face="normal" font="default" size="100%">*Protein Folding</style></keyword><keyword><style  face="normal" font="default" size="100%">*Sequence Homology</style></keyword><keyword><style  face="normal" font="default" size="100%">Amino Acid</style></keyword><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein Structure</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2006</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/1471-2105-7-455.pdf</style></url></related-urls></urls><pub-location><style face="normal" font="default" size="100%">Department of Computer Science &amp; Engineering, University of Minnesota, Minneapolis, Minnesota, USA. rangwala@cs.umn.edu</style></pub-location><volume><style face="normal" font="default" size="100%">7</style></volume><pages><style face="normal" font="default" size="100%">455</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">BACKGROUND: Protein remote homology detection and fold recognition are central problems in computational biology. Supervised learning algorithms based on support vector machines are currently one of the most effective methods for solving these problems. These methods are primarily used to solve binary classification problems and they have not been extensively used to solve the more general multiclass remote homology prediction and fold recognition problems. RESULTS: We present a comprehensive evaluation of a number of methods for building SVM-based multiclass classification schemes in the context of the SCOP protein classification. These methods include schemes that directly build an SVM-based multiclass model, schemes that employ a second-level learning approach to combine the predictions generated by a set of binary SVM-based classifiers, and schemes that build and combine binary classifiers for various levels of the SCOP hierarchy beyond those defining the target classes. CONCLUSION: Analyzing the performance achieved by the different approaches on four different datasets we show that most of the proposed multiclass SVM-based classification approaches are quite effective in solving the remote homology prediction and fold recognition problems and that the schemes that use predictions from binary models constructed for ancestral categories within the SCOP hierarchy tend to not only lead to lower error rates but also reduce the number of errors in which a superfamily is assigned to an entirely different fold and a fold is predicted as being from a different SCOP class. Our results also show that the limited size of the training data makes it hard to learn complex second-level models, and that models of moderate complexity lead to consistently better results.</style></abstract><custom1><style face="normal" font="default" size="100%">Cited:</style></custom1><custom2><style face="normal" font="default" size="100%">22</style></custom2></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>47</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">Eric Lantz</style></author><author><style face="normal" font="default" size="100%">Roy Musselman</style></author><author><style face="normal" font="default" size="100%">Kurt Pinnow</style></author><author><style face="normal" font="default" size="100%">Brian Smith</style></author><author><style face="normal" font="default" size="100%">Brian Wallenfelt</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Massively parallel BLAST for the Blue Gene/L</style></title><secondary-title><style face="normal" font="default" size="100%">In High Availability and Performance Computing Workshop</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2005</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/10.1.1.96.8496.pdf</style></url></related-urls></urls><language><style face="normal" font="default" size="100%">eng</style></language></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Huzefa Rangwala</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Profile-based direct kernels for remote homology detection and fold recognition.</style></title><secondary-title><style face="normal" font="default" size="100%">Bioinformatics</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">Algorithms</style></keyword><keyword><style  face="normal" font="default" size="100%">Amino Acid Sequence</style></keyword><keyword><style  face="normal" font="default" size="100%">Artificial Intelligence</style></keyword><keyword><style  face="normal" font="default" size="100%">Automated</style></keyword><keyword><style  face="normal" font="default" size="100%">Automatic Data Processing</style></keyword><keyword><style  face="normal" font="default" size="100%">Cluster Analysis</style></keyword><keyword><style  face="normal" font="default" size="100%">Computational Biology/*methods</style></keyword><keyword><style  face="normal" font="default" size="100%">Computer Simulation</style></keyword><keyword><style  face="normal" font="default" size="100%">Computing Methodologies</style></keyword><keyword><style  face="normal" font="default" size="100%">Databases</style></keyword><keyword><style  face="normal" font="default" size="100%">hf11</style></keyword><keyword><style  face="normal" font="default" size="100%">Models</style></keyword><keyword><style  face="normal" font="default" size="100%">Pattern Recognition</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein Conformation</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein Folding</style></keyword><keyword><style  face="normal" font="default" size="100%">Protein/methods</style></keyword><keyword><style  face="normal" font="default" size="100%">Proteins</style></keyword><keyword><style  face="normal" font="default" size="100%">Sequence Alignment</style></keyword><keyword><style  face="normal" font="default" size="100%">Sequence Analysis</style></keyword><keyword><style  face="normal" font="default" size="100%">Software</style></keyword><keyword><style  face="normal" font="default" size="100%">Statistical</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2005</style></year></dates><urls><related-urls><url><style face="normal" font="default" size="100%">http://www.cs.gmu.edu/~hrangwal/sites/default/files/Bioinformatics-2005-Rangwala-4239-47.pdf</style></url></related-urls></urls><number><style face="normal" font="default" size="100%">23</style></number><pub-location><style face="normal" font="default" size="100%">Department of Computer Science and Engineering, University of Minnesota Minneapolis, MN 55455, USA.</style></pub-location><volume><style face="normal" font="default" size="100%">21</style></volume><pages><style face="normal" font="default" size="100%">4239‚Äì4247</style></pages><language><style face="normal" font="default" size="100%">eng</style></language><abstract><style face="normal" font="default" size="100%">MOTIVATION: Protein remote homology detection is a central problem in computational biology. Supervised learning algorithms based on support vector machines are currently one of the most effective methods for remote homology detection. The performance of these methods depends on how the protein sequences are modeled and on the method used to compute the kernel function between them. RESULTS: We introduce two classes of kernel functions that are constructed by combining sequence profiles with new and existing approaches for determining the similarity between pairs of protein sequences. These kernels are constructed directly from these explicit protein similarity measures and employ effective profile-to-profile scoring schemes for measuring the similarity between pairs of proteins. Experiments with remote homology detection and fold recognition problems show that these kernels are capable of producing results that are substantially better than those produced by all of the existing state-of-the-art SVM-based methods. In addition, the experiments show that these kernels, even when used in the absence of profiles, produce results that are better than those produced by existing non-profile-based schemes. AVAILABILITY: The programs for computing the various kernel functions are available on request from the authors.</style></abstract></record><record><source-app name="Biblio" version="6.x">Drupal-Biblio</source-app><ref-type>17</ref-type><contributors><authors><author><style face="normal" font="default" size="100%">Mayer, Benjamin W</style></author><author><style face="normal" font="default" size="100%">Rangwala, Huzefa S</style></author><author><style face="normal" font="default" size="100%">Gupta, Rohit</style></author><author><style face="normal" font="default" size="100%">Srivastava, Jaideep</style></author><author><style face="normal" font="default" size="100%">George Karypis</style></author><author><style face="normal" font="default" size="100%">Kumar, Vipin</style></author><author><style face="normal" font="default" size="100%">de Groen, Piet C</style></author></authors></contributors><titles><title><style face="normal" font="default" size="100%">Feature mining for prediction of degree of liver fibrosis.</style></title><secondary-title><style face="normal" font="default" size="100%">AMIA Annu Symp Proc</style></secondary-title></titles><keywords><keyword><style  face="normal" font="default" size="100%">*Diagnosis</style></keyword><keyword><style  face="normal" font="default" size="100%">Clinical</style></keyword><keyword><style  face="normal" font="default" size="100%">Computer-Assisted</style></keyword><keyword><style  face="normal" font="default" size="100%">Decision Support Systems</style></keyword><keyword><style  face="normal" font="default" size="100%">Humans</style></keyword><keyword><style  face="normal" font="default" size="100%">Liver Cirrhosis/*diagnosis</style></keyword></keywords><dates><year><style  face="normal" font="default" size="100%">2005</style></year></dates><pub-location><style face="normal" font="default" size="100%">University of Minnesota, Minneapolis, MN, USA.</style></pub-location><pages><style face="normal" font="default" size="100%">1048</style></pages><language><style face="normal" font="default" size="100%">eng</style></language></record></records></xml>