<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
 
 <!-- If IE use the latest rendering engine -->
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
  
  <!-- Set the page to the width of the device and set the zoon level -->
  <meta name="viewport" content="width = device-width, initial-scale = 1">
  <title>Huzefa Rangwala @ Computer Science, George Mason University</title>
  <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
   
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
   <style>
   .jumbotron{
        /* background-color:#2E2D88;
             color:white; */
   }
   /* Adds borders for tabs */
   .tab-content {
         border-left: 1px solid #ddd;
             border-right: 1px solid #ddd;
                 border-bottom: 1px solid #ddd;
                     padding: 10px;
   }
   .nav-tabs {
         margin-bottom: 0;
   }
   </style>
    
    </head>
    <body>
    

     <!--Navigation-->
    <!--Navigation bar-->
    <div class="container-fluid" id="nav-placeholder">

    </div>

    <script type="text/javascript">
    $(function(){
        $("#nav-placeholder").load("menus/topnav.html");
                         
    });
    </script>
    <!--end of Navigation bar-->

     
     <!-- CONTAINERS -->
     <!-- container puts padding around itself while container-fluid fills the whole screen. Bootstap grids require a container. -->
     <div class="container">
        <div class="row">
            <div class="col-10">
            <div class="jumbotron">
                <table>
                <tr>  <td>
                <h4 align="center">  CAREER: Annotating the Microbiome using Machine Learning Methods </h4>
                <br>
                This project addresses an important challenge of developing sophisticated and novel machine learning techniques for complex real-world problems. New technologies allow us to determine the genomes of organisms co-existing within various ecosystems ranging from ocean, soil and human-body. Several researchers have embarked on studying the pathogenic role played by the microbiome, defined as the collection of microbial organisms within the human body, with respect to human health and disease conditions.  <br>
                <br>
The research activities in this CAREER project will develop approaches for the identification of taxonomy, function and metabolic potential from the collective genomes samples. A key contribution will be the development of multi-task learning approaches that combine information across multiple hierarchical databases associated with the annotation problems. During research, the PI will investigate the best ways to capture the underlying hierarchical structure, prevalent within different annotation databases. The rationale underlying this proposed research is that there is a wealth of complementary information that exists across several manually curated biological databases. Associating microbiome with phenotype requires integration of various high-throughput omic data sources (genomic, metabolic, proteomic) that may not be uniformly available across all samples. The PI will develop data fusion classifiers within the multi-task learning paradigm to integrate heterogeneous, incomplete data sources for predicting phenotypes. This project will lead to the following key contributions: (i) Improved metagenome annotation models by integration of multiple prediction tasks and associated databases. (ii) Incorporation of hierarchical information within regularized multi-task learning. (iii) Integration of diverse and incomplete information sources. (iv) Scalable algorithms that use hash based feature representations and improve the learning rates. <br>
<br>

This project is interdisciplinary and spans the fields of machine learning, bioinformatics, metagenomics, microbiology and environmental ecology. This project will foster the the synergy between teaching and research by providing an environment for all students to develop intellectually and professionally. The project integrates the research with an education plan focused on mentoring of high school, undergraduate and graduate students, curriculum development and laboratory visits. Planned activities include training of inter-disciplinary researchers, integration of microbiome analysis related projects within the classes, curriculum enhancement and implementation of new learning strategies. Open source software and tools will be developed as part of this project, that will enhance scientific understanding and discovery amongst a broad and diverse group of researchers.


		</td></tr>
                </table>
                <hr class="my-4">
                <h4> Funded by NSF CAREER . See Abstract for Award <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1252318&HistoricalAwards=false">here </a> </h4>

                <hr class="my-4">

                <h4> Investigators: </h4>
                <ul class="list-group"> <li class="list-group-item">  <a href="home.html"> Huzefa Rangwala </a>  (Computer Science) </li> 
                </ul>

                
                <div class="row">
                  <div class="col-md-4">
                 <div class="panel panel-default">
                 <div class="panel-heading">Ph.D. Students</div>
                 <div class="panel-body">
                
                 Mohammad Arifur Rahman <br>
		         Azad Naik, Ph.D., Now at Microsoft <br>
	             Anveshi Charuvaka, Ph.D., Now at GE Digital <br>
		         Sam Blasiak, Ph.D., Now at Google, Inc <br>
		         Zeehasham Rasheed, Now at AOL, Inc. <br>
                 </div>
                 </div>
                 </div>
                  <div class="col-md-4">
                 <div class="panel panel-default">
                 <div class="panel-heading">M.S. Students</div>
                 <div class="panel-body">
                   Syed Faraz Mahmood, Now at Nodality <br>
                </div>
                 </div>
                 </div>
                 <div class="col-md-4">
                 <div class="panel panel-default">
                 <div class="panel-heading">B.S. Students</div>
                 <div class="panel-body">
                  Nathan LaPierre, Now Ph.D. student at UCLA <br>
                  Anisha Kolla <br>
                  Eric Tao (High School Student) <br>
		          Ashwin Sekar (High School Student)
		
		</div>
                </div>
                </div>
               </div>

               <div class="row">
                <div class="col-10">
                <h4 align="center"> Products: </h4> 
                <h5> Software: </h5>
                <ul class="list-group">
                <li class="list-group-item"> 
                  <a href="http://cs.gmu.edu/~mlbio/MC-MinH/">  MC-MinH: </a> Metagenome Clustering using Minwise based Hashing
                  
                    </li>
                 
                <li class="list-group-item"> 
                 <a href="http://cs.gmu.edu/~mlbio/MrMC-MinH/">MrMC-MinH: </a> Map-Reduce based Metagenome Clustering using Minwise based Hashing

                </li>
                
                <li class="list-group-item">
                <a href="https://cs.gmu.edu/~mlbio/LSH-DIV/"> LSH-Div </a>  Species Diversity Estimation using Locality Sensitive Hashing </li>

                <li class="list-group-item">
                                      <a href="https://cs.gmu.edu/~mlbio/TAC-ELM/"> TAC-ELM: </a> Taxonomic Classification with Extreme Learning Machines
                 </li> 
                 </ul>
                <h5> Selected Publications: </h5>
                <ul class="list-group">
                  <li class="list-group-item"> Mohammad Arifur Rahman, Nathan Lapierre, Huzefa Rangwala, and Daniel Barbara. Clustering
                   metagenome sequences using canopies. In Proceedings of the 9th International Conference on Bioinformatics and Computational Biology, pages 1-8. ISCA, 2017.
                   </li>
            <li class="list-group-item ">Nathan LaPierre, Mohammad Arifur Rahman, and Huzefa Rangwala. Camil: Clustering and assembly  with multiple instance learning for phenotype prediction. In Bioinformatics and Biomedicine (BIBM), IEEE International Conference on, pages 33-40. IEEE, 2016. Acceptance Rate: 69/361 = 19%
            </li>


            <li class="list-group-item ">
            Azad Naik and Huzefa Rangwala. Embedding feature selection for large-scale hierarchical classification. Proceedings of the 2016 IEEE International Conference on Big Data, 2016. Acceptance Rate: 79/423
            = 18.68%
            </li>
            
            <li class="list-group-item"> Nuttachat Wisittipanit, Huzefa Rangwala, Masoumeh Sikaroodi, ECE Mutlu, Ali Keshavarzian, and
                       Patrick Gillevet. Classification methods for the analysis of lh-pcr data associated with inflammatory
                                  bowel disease patients. International Journal of Bioinformatics Research and Applications, 11(2):111-129, 2015 

<li class="list-group-item">
          Huzefa Rangwala, Anveshi Charuvaka, and Zeehasham Rasheed. Machine learning approaches for
                    metagenomics. In Proceedings of the European Conference on Machine Learning and Principles of
                              Knowledge Discovery and Data Mining (ECML-PKDD), Nancy, France, September 2014. ECML. Nectar Track Acceptance Rate: 10/22 = 45.4% 
                                         </li>



                                         <li class="list-group-item">
                                                 Anveshi Charuvaka and Huzefa Rangwala. Classifying protein sequences using regularized multitask learning. IEEE/ACM Transactions on Computational Biology and Bioinformatics, pages 1-14, 2014. IF: 2.25. 
                                                            </li>

                                                            <li class="list-group-item">
                                                                    Guoxian Yu, Huzefa Rangwala, Carlotta Domeniconi, Guoji Zhang, and Zili Zhang. Predicting protein functions using multiple kernels. IEEE/ACM Transactions on Computational Biology and Bioinformatics, pages 1-14, 2014. IF: 2.25 
                                                                               </li>

                                                                               <li class="list-group-item">
                                                                                       Guoxian Yu, Huzefa Rangwala, Carlotta Domeniconi, Guoji Zhang, and Zhiwen Yu. Protein function prediction with incomplete annotations. IEEE/ACM Transactions on Computational Biology and Bioinformatics, 11(3):579-591, 2014. IF: 2.25 
                                                                                                  </li>


<li class="list-group-item">     Guoxian Yu, Huzefa Rangwala, Carlotta Domeniconi, Guoji Zhang, and Zhiwen Yu. Protein function prediction using multi-label ensemble classification. ACM/IEEE Transactions on Computational Biology and Bioinformatics, 10(4):1045-1057, 2013. IF: 2.25. 
           </li>



                   <li class="list-group-item">   Zeehasham Rasheed, Huzefa Rangwala, and Daniel Barbara. 16s rrna metagenome clustering and diversity estimation using locality sensitive hashing. BMC Systems Biology, 7(S-4)(S11), 2013. IF:  2.98. 
                              </li>


 <li class="list-group-item">   Jasmohan S Bajaj, Douglas M Heuman, Arun J Sanyal, Phillip B Hylemon, Richard K Sterling, R Todd Stravitz, Michael Fuchs, Jason M Ridlon, Kalyani Daita, Pamela Monteith, Nicole A Noble, Melanie B White, Andmorgan Fisher, Masoumeh
  Sikaroodi, Huzefa Rangwala, and Patrick M Gillevet. Modulation of the metabiome by rifaximin in patients with cirrhosis and minimal hepatic encephalopathy. PloS one, 8(4):e60042, 2013. IF: 4.09. 
             </li>

             <li class="list-group-item">        Sam Blasiak, Huzefa Rangwala, and Kathryn B. Laskey. Relevant subsequence detection with sparse dictionary learning. In Proceedings of the European Conference on Machine Learning and Principles  of Knowledge Discovery and Data Mining (ECML-PKDD), Prague, CZ, September 2013. ECML. Acceptance Rate: 111/443 = 25.0%. 
                        </li>

                          <li class="list-group-item"> Guoxian Yu, Carlotta Domeniconi, Huzefa Rangwala, and Guoji Zhang. Protein function prediction  using dependence maximization. In Proceedings of the European Conference on Machine Learning and  Principles of Knowledge Discovery and Data Mining (ECML-PKDD), Prague, CZ, September 2013.  ECML. Acceptance Rate: 111/443 = 25.0% 
                                     </li>


                                       <li class="list-group-item"> Guoxian Yu, Huzefa Rangwala, Carlotta Domeniconi, Guoji Zhang, and Zili Zhang. Protein function prediction by integrating multiple kernels. In 23rd. International Joint Conference on Artificial Intelligence, pages 1-7, Beijing, China, August 2013. AAAI. Acceptance Rate: 413/1473 = 28% 
                                                  </li>

<li class="list-group-item"> Zeehasham Rasheed and Huzefa Rangwala. Mc-minh: Metagenome clustering using minwise based  hashing. In SIAM International Conference in Data Mining (SDM), Austin, TX, May 2013. SIAM.  Acceptance Rate: 89/348 = 25.5%. 
            </li>



                
                </ul>
                </div>
                </div>

         </div>
      </div>
                                                                                                                                                    
                                                                                                                                                                                      



         </script>
         <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>

     </body>
     </html>
