<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
 
 <!-- If IE use the latest rendering engine -->
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
  
  <!-- Set the page to the width of the device and set the zoon level -->
  <meta name="viewport" content="width = device-width, initial-scale = 1">
  <title>Huzefa Rangwala @ Computer Science, George Mason University</title>
  <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
   
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
   <style>
   .jumbotron{
        /* background-color:#2E2D88;
             color:white; */
   }
   /* Adds borders for tabs */
   .tab-content {
         border-left: 1px solid #ddd;
             border-right: 1px solid #ddd;
                 border-bottom: 1px solid #ddd;
                     padding: 10px;
   }
   .nav-tabs {
         margin-bottom: 0;
   }
   </style>
    
    </head>
    <body>
    

     <!--Navigation-->
    <!--Navigation bar-->
    <div class="container-fluid" id="nav-placeholder">

    </div>

    <script type="text/javascript">
    $(function(){
        $("#nav-placeholder").load("menus/topnav.html");
    });
    </script>
    <!--end of Navigation bar-->

     
     <!-- CONTAINERS -->
     <!-- container puts padding around itself while container-fluid fills the whole screen. Bootstap grids require a container. -->
     <div class="container">
        <div class="row">
            <div class="col-10">

         My research has lead to the development of several software and web servers. These are made available to the academic research community. 

        <div class="panel panel-info">
            <div class="panel-heading"> DMGrader: Data Mining Grader </div> 
            <div class="panel-body"><a href="https://github.com/rangwala/dmgrader">DMGrader: </a> is a  customizable web-based data analytics competition hosting framework written in Django and Python. Rangwala has used this in teaching data science at George Mason University and hosting data analytics hackathon. 
            This software is under continuous development. 
            
            </div>
        </div>
        <div class="panel panel-info">
            <div class="panel-heading"> NTSGP: Next-Term Student Grade Prediction Toolkit </div>
            <div class="panel-body"> <a href="https://github.com/macks22/ntsgp"> NTSGP </a> funded by NSF BIG Data project No. 1447489 is a comprehensive set of Python-based programs for implementing an educational recommender system. Please see paper <a href="http://www.educationaldatamining.org/JEDM/index.php/JEDM/article/view/JEDM2016-8-1-3"> here </a> for details. 
            </div>
            </div>
       <div class="panel panel-info">
            <div class="panel-heading"> HierCost: Hierarchical Cost Sensitive Learning </div>
            <div class="panel-body"> <a href="https://cs.gmu.edu/~mlbio/HierCost/">HierCost</a> toolkit is a set of programs for supervised  classification for single-label and multi-label hierarchical classification using cost sensitive logistic regression based classifier written in python.
            </div>.
            </div>

         
         <div class="panel panel-info">
            <div class="panel-heading"> MC-MinH and MrMC-MinH: Metagenome Clustering using Minwise based Hashing</div>
            <div class="panel-body">
            <a href="http://cs.gmu.edu/~mlbio/MC-MinH/">  MC-MinH </a> is a metagenome analysis toolkit. MC-MinH algorithm uses the min-wise hashing approach, along with a greedy clustering algorithm to group 16S and whole metagenomic sequences. We represent unequal length sequences using contiguous subsequences or k-mers, and then approximate the computation of pairwise similarity using independent min-wise hashing. The algorithm is written in C and is available using the GNU GPL license. <br>
            <br>
           
           <a href="http://cs.gmu.edu/~mlbio/MrMC-MinH/">MrMC-MinH </a> is a Map-Reduce based algorithm for metagenome clustering using minwise hashing. It is an extension of our previously developed, greedy clustering algorithm MC-MinH (http://www.cs.gmu.edu/~mlbio/MC-MinH/). The algorithm is written in Java and Pig programming language. 

            </div>
        </div>


         
         
         <div class="panel panel-info">
            <div class="panel-heading"> LSH-Div: Species Diversity Estimation using Locality Sensitive Hashing </div>
            <div class="panel-body">
            <a href="https://cs.gmu.edu/~mlbio/LSH-DIV/"> LSH-Div </a> is a metagenome analysis toolkit. The clustering algorithm groups sequences into Operational Taxonomic Units (OTUs) using the LSH function within a greedy, iterative clustering framework. LSH-Div reports the standard species richness metrics such as Chao1 Index, Shannon Diversity Index and Abundance-based Coverage Estimator (ACE) Index after assigning sequences within a sample to different OTUs (or clusters)
            </div>
        </div>
        
        <div class="panel panel-info">
            <div class="panel-heading">TAC-ELM: Taxonomic Classification with Extreme Learning Machines
            </div>
            <div class="panel-body">
            <a href="https://cs.gmu.edu/~mlbio/TAC-ELM/"> TAC-ELM </a> is a metagenome analysis toolkit. It is a new taxonomy classification scheme that extracts composition-based features (oligonucleotides and GC content) from the the short sequence reads and develop a neural network-based model. To train the parameters of the model we use an analytical framework, called extreme learning machine (ELM) to learn the parameters of the models.
            </div>
        </div>


         
         <div class="panel panel-info">
            <div class="panel-heading"> svmPRAT: svm-Based Protein Residue Annotation Toolkit</div>
            <div class="panel-body">
            <a href="http://www.cs.gmu.edu/~mlbio/prosat/">svmPRAT: </a> is a general purpose protein residue annotation toolkit  to allow biologists to formulate residue-wise prediction problems. svmPRAT formulates annotation problem as a classification or regression problem using support vector machines. The key features of svmPRAT are its ease of use  to incorporate any user-provided information in the form of feature matrices. For every residue svmPRAT captures local information around the reside to create fixed length feature vectors. svmPRAT implements accurate and fast kernel functions, and also introduces a flexible window-based encoding scheme that allows better capture of signals for certain prediction problems.</td>
            </div>
            </div>
   
         <div class="panel panel-info">
            <div class="panel-heading">MONSTER: Minnesota prOteiN  Sequence annoTation servER </div> 
            <div class="panel-body">
            <a href="http://bio.dtc.umn.edu/monster/">MONSTER </a> is a server for predicting the local structure and function properties  of protein residues. MONSTER provides residue-wise annotation services, that include secondary structure, transmembrane-helix region,  disorder region, protein-DNA binding site, \red {ligand-binding site}, local structure alphabet,  solvent accessibility  surface area, and residue-wise contact order  prediction. MONSTER uses sequence-derived information (in the form of PSI-BLAST profiles), a window-based encoding scheme with  an accurate kernel function to perform the classification or estimation.  The  user provides an amino acid sequence and selects the desired predictions, and submits a job to the MONSTER server. The results are emailed to the user as a link directing the user to a well formatted HTML output page.  </td> </tr> </table>
            </div>
            </div>
        
           <div class="panel panel-info">
            <div class="panel-heading"> MARINER: MinnesotA pRotein modelINg servER </div>  
            <div class="panel-body"> <a href="http://bio.dtc.umn.edu/mariner/"> MARINER: </a>  is a server for predicting the three-dimensional structure of proteins using  homology modeling based techniques. This server is always under development, and was used for participation in the <a href="http://predictioncenter.org"> CASP 8</a> protein structure prediction competition. Watch this space for a future version of this server. Also students at George Mason interested in the competition, please get in touch with me.  </td></tr>
            </div>
            </div>
         
           <div class="panel panel-info">
            <div class="panel-heading"> Profile-based Kernel Compute Package </div> 
            <div class="panel-body">
         <a href="http://bioinfo.cs.umn.edu/supplements/profile-kernels/index.html"> kernel-compute  </a> is a package that computes pairwise profile-based similarity matrix. This matrix can
         then be converted into a valid kernel matrix with an eigen value transformation. This scoring matrix has shown to be the best performing method for developing remote homology detection and fold recognition models.</td>
         </div>
      </div>
                                                                                                                                                    
                                                                                                                                                                                      
    </div>


    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>

     </body>
     </html>
