<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
		<title>Readings | MLBio+ Laboratory</title>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="/~hrangwal/sites/all/themes/nitobe/favicon.ico" type="image/x-icon" />
		<link type="text/css" rel="stylesheet" media="all" href="/~hrangwal/sites/default/files/css/css_eaed5ee67284326ba9e2a752c33bb566.css" />
<link type="text/css" rel="stylesheet" media="screen" href="/~hrangwal/sites/default/files/css/css_a5d34e900b3fb4657272ecfb0d3f0ee7.css" />
<link type="text/css" rel="stylesheet" media="print" href="/~hrangwal/sites/default/files/css/css_1f691132d615fa9edb76248c88e261a9.css" />
<style type="text/css">#masthead{background-image:url(http://www.cs.gmu.edu/~hrangwal/sites/all/themes/nitobe/headers/Earth_from_ISS.jpg);}</style>
		<script type="text/javascript" src="/~hrangwal/sites/default/files/js/js_7f97b92a3f70662cff6fd903d3a092d8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, { "basePath": "/~hrangwal/" });
//--><!]]>
</script>
		<!--[if lt IE 8]>
			<link type="text/css" rel="stylesheet" media="screen" href="/~hrangwal/sites/all/themes/nitobe/styles/fix-ie.css" />
		<![endif]-->
	</head>
	<body class="nitobe not-front not-logged-in page-node node-type-story one-sidebar sidebar-left">
		<div id="page-wrapper" class="clear-block">
			<div id="header-area" class="container-16">
				<div id="title-group" class="grid-12">
			            <a href="/~hrangwal/" title="MLBio+ Laboratory"><img src="/~hrangwal/sites/all/themes/nitobe/logo.png" alt="" id="logo" /></a>			            <span id="site-title"><a href="/~hrangwal/" title="MLBio+ Laboratory">MLBio+<span class="alt-color">Laboratory</span></a></span>			            <span id="site-slogan">Machine Learning in Biomedical Informatics</span>			        </div><!-- /title-group -->
			        			        			        	<div id="search-top" class="grid-4"><form action="/~hrangwal/node/147"  accept-charset="UTF-8" method="post" id="search-theme-form">
<div><div id="search" class="container-inline">
  <div class="form-item" id="edit-search-theme-form-1-wrapper">
 <label for="edit-search-theme-form-1">Search this site: </label>
 <input type="text" maxlength="128" name="search_theme_form" id="edit-search-theme-form-1" size="15" value="" title="Enter the terms you wish to search for." class="form-text" />
</div>
<input type="submit" name="op" id="edit-submit" value="Search"  class="form-submit" />
<input type="hidden" name="form_build_id" id="form-3898b8696d3867c59d73e26cae9f6bc2" value="form-3898b8696d3867c59d73e26cae9f6bc2"  />
<input type="hidden" name="form_id" id="edit-search-theme-form" value="search_theme_form"  />
</div>

</div></form>
</div>
			                  		<hr class="break"/>
          		            		<div id="header-links" class="grid-16">
            		    <ul id="primary-nav" class="grid-16"><li class="menu-143 first"><a href="/~hrangwal/myself" title="Huzefa Rangwala">Home</a></li>
<li class="menu-567"><a href="https://cs.gmu.edu/~hrangwal/blog/" title="">News</a></li>
<li class="menu-498"><a href="/~hrangwal/mlbio" title="">Projects</a></li>
<li class="menu-462"><a href="/~hrangwal/biblio" title="Displays all the Publications">Publications</a></li>
<li class="menu-349"><a href="/~hrangwal/dmrg" title="Data Mining Reading Group">Reading Group</a></li>
<li class="menu-146"><a href="/~hrangwal/node/4" title="Software and Web Servers">Software</a></li>
<li class="menu-305"><a href="/~hrangwal/node/122" title="Service">Service</a></li>
<li class="menu-500"><a href="/~hrangwal/node/91" title="">Members</a></li>
<li class="menu-148"><a href="/~hrangwal/node/7" title="Classes">Teaching</a></li>
<li class="menu-501 last"><a href="/~hrangwal/node/373" title="Contact Information">Contact</a></li>
</ul>            		                  			<hr class="break"/>
            		                		              			</div><!-- #header-links -->
          		          		
		         	
			<div id="masthead" class="grid-16">
			              		</div><!-- /masthead -->
          		<hr class="rule-top grid-16"/>
          		 	
			</div><!-- /header-area -->
			<hr class="break"/>
			<div id="content-area" class="container-16">
				<div id="content" class="grid-12 push-4">
					<div class="breadcrumb"><a href="/~hrangwal/">Home</a></div>																				          				<div id="page-headline" class="clear-block">
    					    <h1 id="page-title" >Readings</h1>    					                      				<span class="timestamp">25 Jan 2010</span>
    					              				</div><!-- #page-headline -->
														<div id="node-147" class="node  node-story clear-block">
  <div class="content clear-block">
        	<div class="node-author">Posted by rangwala</div>
            <span class='print-link'></span><p>The material here is shared and developed by Professor George Karypis, University of Minnesota &amp; Huzefa Rangwala, George Mason University. You are responsible for material under the sections <font color="blue"> Reading Materials </font></p>
<hr />
<p>Here is what we have covered in class so far</p>
<hr />
<p><b> 01.25.2010: Introduction and Course Overview </b><br />
<pr>
<p>Overview of the goals of this course and a brief introduction to some biological concepts that will be used throughout the course. Also, a brief review of some of the CS-related concepts that are required for the course.</p>
<p><b> Readings </b></p>
<p><font color="red"> The first two chapters</font> of the text book. These chapters should be treated as reference material, in the sense that they should be understood (not memorized), especially for students with no biology background. Some of the key concepts introduced here will be revisited throughout the course.</p>
<p>Also, I have made available online some additional background material which should be read if the need arises. They make a very <font color="red"> nice bed-time reading</font>. Note that even though the reading may look long, there is a lot of duplication among the various reading items.</p>
<p><u> Additional Material </u></p>
<ul>
<li> <a href="http://www.cs.gmu.edu/~hrangwal/files/Papers/whatisbio-informatics.pdf"> What is Bioinformatics? </a> </li>
<li><a href="http://www.cs.gmu.edu/~hrangwal/files/Papers/01-Hunter.pdf"> Molecular Biology for Computer Scientists </a></li>
</ul>
<hr />
<p><b>02.01.2010: Sequence Comparison and Alignments</b></p>
<p><b> Motivation for Sequence Comparison </b></p>
<p>Focus on the problem of comparing DNA and amino acid sequences in order to primarily detect homology relationships and to a somewhat lesser extent, functional relationships.  Establish the connection between sequence-similarity-based homology detection with sequence alignment and introduce the various issues involved in aligning two or more biological sequences. </p>
<p><u> <font color="blue"> Reading Material </font></u></p>
<ol>
<li> Sections 4.1-4.4 from text book </li>
<li> Slides </li>
</ol>
<p><b>Algorithms for Pairwise Sequence Alignments </b></p>
<p>Focus on solving the problem of aligning a pair of sequences and cover dynamic programming-based algorithms for global, local, and semi-global sequence alignment. In addition, algorithms to handle the different gap models are discussed and analyzed. </p>
<p><u> <font color="blue"> Reading Material </font> </u></p>
<ol>
<li>
Section 4.4, 5.2 from textbook
</li>
<li><a href="http://www.cs.gmu.edu/~hrangwal/files/Chapter11_Gusfield_.pdf">Chapter 11 from Gusfield ( Core String Edits, Alignments, and Dynamic Programming) </a></li>
</ol>
<hr />
<p><b> 02.08.2010: Multiple Sequence Alignment Algorithms </b></p>
<p>Focus on solving the problem of aligning a set of sequences together and cover various topics related to scoring multiple sequence alignments, representing them, and constructing them using both optimal dynamic-programming based algorithms as well as various heuristic approaches. Some of the heuristic approaches are guaranteed to be within a small constant factor from the optimal alignment, whereas some of the other heuristic approaches focus more on computing multiple sequence alignments that are correct in an evolutionary sense. </p>
<p><font color="blue"> <u> Reading Material </u> </font></p>
<ol>
<li> <a href="http://www.cs.gmu.edu/~hrangwal/files/Chapter14_Gusfield.pdf">Chapter 14.1 to 14.6.1 only from Gusfield </a>
</li>
<li> Chapters 6.4, 6.5 from textbook </li>
<li> Lecture Slides </li>
</ol>
<hr />
<p><b> 02.22.2010:  Database Search </b></p>
<p>Focus on why is database search needed. How could one come up with faster and accurate ways of going through and finding similar sequences. Check out the indexing methods used by Fasta and BLAST.</p>
<p><font color="blue"> <u> Reading Material </u> </font></p>
<ol>
<li> Class Slides </li>
<li> Section 5.3 from your text book </li>
</ol>
<p><u> Additional Material</u></p>
<ul>
<li>
<a href="http://www.cs.gmu.edu/~hrangwal/files/BLAST_altschul_JMB90.pdf">BLAST Paper (Original) </a>
</li>
<li>
<a href="http://www.cs.gmu.edu/~hrangwal/files/FASTA1998.pdf"> FASTA paper </a>
</li>
<li>
<a href="http://www.cs.gmu.edu/~hrangwal/files/Papers/DB-Search/Gap-Blast.pdf"> Gapped BLAST and PSI-BLAST </a></li></ul></pr></p>

<hr />
<p><b> 03.01.2010:  Scoring Matrices </b></p>
<p>Focus on how to derive scoring matrices and provide a probabilistic interpretation for their construction derivation. The overall framework is illustrated by describing how the PAM and BLOSUM matrices have been derived.</p>
<p>Scoring matrices are critical in encoding biological information into the alignment procedures. It is the use of the appropriate scoring matrix that ensures (up to a point) that the highest scoring alignment will actually be able to detect homology by aligning against each other the residues of two proteins that evolved from the same ancestral residue. </p>
<p><font color="blue"> <u> Reading Material </u> </font></p>
<ol>
<li> Chapter 5.1 from textbook </li>
<li> Lecture Slides </li>
</ol>
<p><u> Additional Material </u></p>
<ul>
<li> <a href="http://www.cs.gmu.edu/~hrangwal/files/Papers/Scoring-Matrices/blosum.pdf">BLOSUM Paper </a>
</li>
<li><a href="http://www.cs.gmu.edu/~hrangwal/files/Papers/2004_Primer_Where_did_the_BLOSUM62_alignment_score_matrix_come_from.pdf"> Where do BLOSUM come from ? </a>
</li></ul>
<hr />
<p><b> 03.01.2010: Statistical Significance of Alignment Scores<br />
</b></p>
<p>Focus on how to assess the statistical significance of alignment scores and present and overview of the methods that are commonly used.</p>
<p><font color="blue"> <u> Reading Material </u> </font></p>
<ol>
<li> Lecture Slides </li>
<li> Sections 5.4 &amp; 6.3 from the textbook. </li>
</ol>
<hr />
<p><b>03.01.2010:  Markov Models </b></p>
<p>Focus on probabilistic models for sequence classification and annotation and describe Markov Chains and Hidden Markov models, which are the most widely used Markovian models in bioinformatics.</p>
<p><font color="blue"> <u> Reading Material </u> </font></p>
<ol>
<li> Lecture Slides </li>
<li> <a href="http://www.cs.gmu.edu/~hrangwal/files/Chapter3_Durbin.pdf"> Chapter 3 from Durbin </a>
</li>
<li> Chapter 6.2 from the textbook </li>
</ol>
<hr />
<p><b>03.22.2010+03.29.2010:  Protein Structure Prediction </b></p>
<p>Discuss the secondary structure organization of protein molecules and different ways of determining them from their primary sequence. </p>
<p><font color="blue"> <u> Reading Material </u> </font></p>
<ol>
<li> Lecture Slides </li>
<li> Chapter 11-12 from the textbook  (Skip RNA)</li>
<li> Chapter 13 from the textbook </li>
</ol>
<p><u> Additional Material</u></p>
<ul>
<li> <a href="http://www.cs.gmu.edu/~hrangwal/files/Papers/SSE.tar.gz"> Several Secondary Structure Prediction Papers </a></li>
<li> <a href="http://www.biomedcentral.com/1471-2105/10/439"> svmPRAT: Protein Residue Annotation Toolkit </a> </li>
<li> <a href="http://glaros.dtc.umn.edu/gkhome/fetch/papers/toptmh.pdf"> TOPTMH: Topology Prediction for Transmembrane Alpha-Helices </a>
</li><li> <a href="http://www.cs.gmu.edu/~hrangwal/files/psp.tar.gz"> Papers on Protein Structure Prediction </a>
</li>
</ul>
<hr />
<hr />
<b>04.19.2010:  Gene Prediction </b>
<p>Focus on the problem of finding the genes in an organisms chromosomal DNA sequence. Describe the various sequence-based signals that exists and the various methods that have been developed for gene prediction in prokaryotic and eukaryotic organisms.</p>
<p><font color="blue"> <u> Reading Material </u> </font></p>
<ol>
<li> Class Slides </li>
<li> Chapters 9 and 10 from the textbook </li>
</ol>
<p><u> Additional Material</u></p>
<ul>
<li> Papers on Gene Prediction </li>
</ul>
  </div>
      <div class="meta">
                <div class="links"><ul class="links inline"><li class="print_mail first last"><a href="/~hrangwal/printmail/147" title="Send this page by e-mail." class="print-mail" rel="nofollow">Send to friend</a></li>
</ul></div>
          </div>
  </div> <!-- node -->
				</div><!-- /content -->
									<div id="left-sidebar" class="grid-4 pull-12">
					  <div id="block-menu-menu-classes" class="block block-menu block-left">
    	<h2>Classes</h2>
    <ul class="menu"><li class="leaf first"><a href="/~hrangwal/cs367" title="CS 367: Computer Systems and Programming">CS 367 (Fall 2017)</a></li>
<li class="expanded last active-trail"><a href="/~hrangwal/node/7" title="">Archived Classes</a><ul class="menu"><li class="leaf first"><a href="/~hrangwal/cs584F2016" title="CS 584: Data Mining (Syllabus Page)">CS 584</a></li>
<li class="collapsed"><a href="/~hrangwal/cs584F2015" title="CS 659: Theory and Applications of Data Mining">CS 584 (TASC)</a></li>
<li class="collapsed"><a href="/~hrangwal/cs695Sp15SocNet" title="CS 695: Social and Information Network Analysis">CS 695 (Spring 2015)</a></li>
<li class="collapsed"><a href="/~hrangwal/cs465F2014" title="CS 465 (Fall 2014)">CS 465 (Fall 2014)</a></li>
<li class="collapsed"><a href="/~hrangwal/CS635Spring2014" title="CS 635: Foundations of Parallel Computation">CS 635 (Spring 2014)</a></li>
<li class="collapsed"><a href="/~hrangwal/cs465-fall2012" title="CS 465: Computer Systems and Architecture (Fall 2012)">CS 465 (Fall 2012)</a></li>
<li class="collapsed"><a href="/~hrangwal/infs755" title="INFS 755: Data Mining">INFS 755 (Fall 2011)</a></li>
<li class="collapsed"><a href="/~hrangwal/cs465F2013" title="CS 465: Computer Systems and Architecture (Fall 2013)">CS 465 (Fall 2013)</a></li>
<li class="collapsed"><a href="/~hrangwal/cs750" title="CS 750: Theory and Applications of Data Mining (Spring 2011)">CS 750 (Spring 2011)</a></li>
<li class="collapsed"><a href="/~hrangwal/cs465" title="CS 465 (Fall 2010) Computer Systems Architecture">CS 465 (Fall 2010)</a></li>
<li class="expanded active-trail"><a href="/~hrangwal/node/20" title="Syllabus">CS 795 (Spring 2010)</a><ul class="menu"><li class="leaf first"><a href="/~hrangwal/node/144" title="Announcements">Announcements</a></li>
<li class="leaf active-trail"><a href="/~hrangwal/node/147" title="Readings" class="active">Readings</a></li>
<li class="leaf"><a href="/~hrangwal/node/143" title="Schedule">Schedule</a></li>
<li class="leaf"><a href="/~hrangwal/node/51" title="Class Slides">Class Slides</a></li>
<li class="leaf"><a href="/~hrangwal/node/148" title="Assignments">Assignments</a></li>
<li class="leaf"><a href="/~hrangwal/forum/5" title="">Class Forum</a></li>
<li class="leaf"><a href="/~hrangwal/node/53" title="Project Ideas and Listings.">Project</a></li>
<li class="leaf last"><a href="/~hrangwal/node/52" title="Resources">Resources</a></li>
</ul></li>
<li class="collapsed"><a href="/~hrangwal/bioml" title="CS 795 (Fall 2009) Biological Data Mining">CS 795 (Fall 2009)</a></li>
<li class="collapsed"><a href="/~hrangwal/class-bio" title="CS 795 (Spring 2009)">CS 795 (Spring 2009)</a></li>
<li class="collapsed last"><a href="/~hrangwal/node/8" title="INFS 755 Data Mining (Fall 2008)">INFS 755 (Fall 2008)</a></li>
</ul></li>
</ul></div><!-- /block-menu-menu-classes -->
<hr class="break"/>					</div><!-- /left-sidebar -->
											</div><!-- /content-area -->
			<hr class="break"/>
			<div id="bottom-blocks" class="container-16">
				<hr class="rule-bottom grid-16"/>
				<div id="bottom-left" class="grid-4">
									</div><!-- /bottom-left -->
				<div id="bottom-center-left" class="grid-4">
									</div><!-- /bottom-center-left -->
				<div id="bottom-center-right" class="grid-4">
									</div><!-- /bottom-center-right -->
				<div id="bottom-right" class="grid-4">
					<div id="block-system-0" class="block block-system block-bottom_right">
    <a href="http://drupal.org"><img src="/~hrangwal/misc/powered-blue-80x15.png" alt="Powered by Drupal, an open source content management system" title="Powered by Drupal, an open source content management system" width="80" height="15" /></a></div><!-- /block-system-0 -->
<hr class="break"/>				</div><!-- /bottom-right -->
			</div><!-- /bottom-blocks -->
			<hr class="break"/>
			<div id="footer-area" class="container-16">
				<hr class="rule-top grid-16"/>
				<div id="footer" class="grid-16">
					(c) Rangwala 2008, George Mason University, Fairfax, VA				</div><!-- /footer -->
			</div><!-- /footer-area -->
			<hr class="break"/>
		</div><!-- /page-wrapper -->
			</body>
</html>
