<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
		<title>NYT article on  | MLBio+ Laboratory</title>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="/~hrangwal/sites/all/themes/nitobe/favicon.ico" type="image/x-icon" />
		<link type="text/css" rel="stylesheet" media="all" href="/~hrangwal/sites/default/files/css/css_eaed5ee67284326ba9e2a752c33bb566.css" />
<link type="text/css" rel="stylesheet" media="screen" href="/~hrangwal/sites/default/files/css/css_a5d34e900b3fb4657272ecfb0d3f0ee7.css" />
<link type="text/css" rel="stylesheet" media="print" href="/~hrangwal/sites/default/files/css/css_1f691132d615fa9edb76248c88e261a9.css" />
<style type="text/css">#masthead{background-image:url(http://www.cs.gmu.edu/~hrangwal/sites/all/themes/nitobe/headers/Earth_from_ISS.jpg);}</style>
		<script type="text/javascript" src="/~hrangwal/sites/default/files/js/js_7f97b92a3f70662cff6fd903d3a092d8.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, { "basePath": "/~hrangwal/" });
//--><!]]>
</script>
		<!--[if lt IE 8]>
			<link type="text/css" rel="stylesheet" media="screen" href="/~hrangwal/sites/all/themes/nitobe/styles/fix-ie.css" />
		<![endif]-->
	</head>
	<body class="nitobe not-front not-logged-in page-node node-type-forum no-sidebars">
		<div id="page-wrapper" class="clear-block">
			<div id="header-area" class="container-16">
				<div id="title-group" class="grid-12">
			            <a href="/~hrangwal/" title="MLBio+ Laboratory"><img src="/~hrangwal/sites/all/themes/nitobe/logo.png" alt="" id="logo" /></a>			            <span id="site-title"><a href="/~hrangwal/" title="MLBio+ Laboratory">MLBio+<span class="alt-color">Laboratory</span></a></span>			            <span id="site-slogan">Machine Learning in Biomedical Informatics</span>			        </div><!-- /title-group -->
			        			        			        	<div id="search-top" class="grid-4"><form action="/~hrangwal/node/367"  accept-charset="UTF-8" method="post" id="search-theme-form">
<div><div id="search" class="container-inline">
  <div class="form-item" id="edit-search-theme-form-1-wrapper">
 <label for="edit-search-theme-form-1">Search this site: </label>
 <input type="text" maxlength="128" name="search_theme_form" id="edit-search-theme-form-1" size="15" value="" title="Enter the terms you wish to search for." class="form-text" />
</div>
<input type="submit" name="op" id="edit-submit" value="Search"  class="form-submit" />
<input type="hidden" name="form_build_id" id="form-433e777ef9d1dafc1e30e09472757062" value="form-433e777ef9d1dafc1e30e09472757062"  />
<input type="hidden" name="form_id" id="edit-search-theme-form" value="search_theme_form"  />
</div>

</div></form>
</div>
			                  		<hr class="break"/>
          		            		<div id="header-links" class="grid-16">
            		    <ul id="primary-nav" class="grid-16"><li class="menu-143 first"><a href="/~hrangwal/myself" title="Huzefa Rangwala">Home</a></li>
<li class="menu-567"><a href="https://cs.gmu.edu/~hrangwal/blog/" title="">News</a></li>
<li class="menu-498"><a href="/~hrangwal/mlbio" title="">Projects</a></li>
<li class="menu-462"><a href="/~hrangwal/biblio" title="Displays all the Publications">Publications</a></li>
<li class="menu-349"><a href="/~hrangwal/dmrg" title="Data Mining Reading Group">Reading Group</a></li>
<li class="menu-146"><a href="/~hrangwal/node/4" title="Software and Web Servers">Software</a></li>
<li class="menu-305"><a href="/~hrangwal/node/122" title="Service">Service</a></li>
<li class="menu-500"><a href="/~hrangwal/node/91" title="">Members</a></li>
<li class="menu-148"><a href="/~hrangwal/node/7" title="Classes">Teaching</a></li>
<li class="menu-501 last"><a href="/~hrangwal/node/373" title="Contact Information">Contact</a></li>
</ul>            		                  			<hr class="break"/>
            		                		              			</div><!-- #header-links -->
          		          		
		         	
			<div id="masthead" class="grid-16">
			              		</div><!-- /masthead -->
          		<hr class="rule-top grid-16"/>
          		 	
			</div><!-- /header-area -->
			<hr class="break"/>
			<div id="content-area" class="container-16">
				<div id="content" class="grid-16">
					<div class="breadcrumb"><a href="/~hrangwal/">Home</a> » <a href="/~hrangwal/forum">Forums</a> » <a href="/~hrangwal/forum/6">CS 465 Discussion Forum</a> » <a href="/~hrangwal/forum/8">CS 750 Discussion Forum</a></div>																				          				<div id="page-headline" class="clear-block">
    					    <h1 id="page-title" >NYT article on </h1>    					                      				<span class="timestamp">02 Dec 2011</span>
    					              				</div><!-- #page-headline -->
														<div id="node-367" class="node  node-forum clear-block">
  <div class="content clear-block">
        	<div class="node-author">Posted by student</div>
            <span class='print-link'></span><p><a href="http://www.nytimes.com/2011/12/01/business/dna-sequencing-caught-in-deluge-of-data.html" title="http://www.nytimes.com/2011/12/01/business/dna-sequencing-caught-in-deluge-of-data.html">http://www.nytimes.com/2011/12/01/business/dna-sequencing-caught-in-delu...</a></p>
<p>DNA Sequencing Caught in Deluge of Data<br />
By ANDREW POLLACK<br />
BGI, based in China, is the world’s largest genomics research institute, with 167 DNA sequencers producing the equivalent of 2,000 human genomes a day.</p>
<p>BGI churns out so much data that it often cannot transmit its results to clients or collaborators over the Internet or other communications lines because that would take weeks. Instead, it sends computer disks containing the data, via FedEx.</p>
<p>“It sounds like an analog solution in a digital age,” conceded Sifei He, the head of cloud computing for BGI, formerly known as the Beijing Genomics Institute. But for now, he said, there is no better way.</p>
<p>The field of genomics is caught in a data deluge. DNA sequencing is becoming faster and cheaper at a pace far outstripping Moore’s law, which describes the rate at which computing gets faster and cheaper.</p>
<p>The result is that the ability to determine DNA sequences is starting to outrun the ability of researchers to store, transmit and especially to analyze the data.</p>
<p>“Data handling is now the bottleneck,” said David Haussler, director of the center for biomolecular science and engineering at the University of California, Santa Cruz. “It costs more to analyze a genome than to sequence a genome.”</p>
<p>That could delay the day when DNA sequencing is routinely used in medicine. In only a year or two, the cost of determining a person’s complete DNA blueprint is expected to fall below $1,000. But that long-awaited threshold excludes the cost of making sense of that data, which is becoming a bigger part of the total cost as sequencing costs themselves decline.</p>
<p>“The real cost in the sequencing is more than just running the sequencing machine,” said Mark Gerstein, professor of biomedical informatics at Yale. “And now that is becoming more apparent.”</p>
<p>But the data challenges are also creating opportunities. There is demand for people trained in bioinformatics, the convergence of biology and computing. Numerous bioinformatics companies, like SoftGenetics, DNAStar, DNAnexus and NextBio, have sprung up to offer software and services to help analyze the data. EMC, a maker of data storage equipment, has found life sciences a fertile market for products that handle large amounts of information. BGI is starting a journal, GigaScience, to publish data-heavy life science papers.</p>
<p>“We believe the field of bioinformatics for genetic analysis will be one of the biggest areas of disruptive innovation in life science tools over the next few years,” Isaac Ro, an analyst at Goldman Sachs, wrote in a recent report.</p>
<p>Sequencing involves determining the order of the bases, the chemical units represented by the letters A, C, G and T, in a stretch of DNA. The cost has plummeted, particularly in the last four years, as new techniques have been introduced.</p>
<p>The cost of sequencing a human genome — all three billion bases of DNA in a set of human chromosomes — plunged to $10,500 last July from $8.9 million in July 2007, according to the National Human Genome Research Institute.</p>
<p>That is a decline by a factor of more than 800 over four years. By contrast, computing costs would have dropped by perhaps a factor of four in that time span.</p>
<p>The lower cost, along with increasing speed, has led to a huge increase in how much sequencing data is being produced. World capacity is now 13 quadrillion DNA bases a year, an amount that would fill a stack of DVDs two miles high, according to Michael Schatz, assistant professor of quantitative biology at the Cold Spring Harbor Laboratory on Long Island.</p>
<p>There will probably be 30,000 human genomes sequenced by the end of this year, up from a handful a few years ago, according to the journal Nature. And that number will rise to millions in a few years.</p>
<p>In a few cases, human genomes are being sequenced to help diagnose mysterious rare diseases and treat patients. But most are being sequenced as part of studies. The federally financed Cancer Genome Atlas, for instance, is sequencing the genomes of thousands of tumors and of healthy tissue from the same people, looking for genetic causes of cancer.</p>
<p>One near victim of the data explosion has been a federal online archive of raw sequencing data. The amount stored has more than tripled just since the beginning of the year, reaching 300 trillion DNA bases and taking up nearly 700 trillion bytes of computer memory.</p>
<p>Straining under the load and facing budget constraints, federal officials talked earlier this year about shutting the archive, to the dismay of researchers. It will remain open, but certain big sequencing projects will now have to pay to store their data there.</p>
<p>If the problem is tough for human genomes, it is far worse for the field known as metagenomics. This involves sequencing the DNA found in a particular environment, like a sample of soil or the human gut. The idea is to take a census of what microbial species are present.</p>
<p>E. Virginia Armbrust, who studies ocean-dwelling microscopic organisms at the University of Washington, said her lab generated 60 billion bases — as much as 20 human genomes — from just two surface water samples. It took weeks to do the sequencing, but nearly two years to then analyze the data, she said.</p>
<p>“There is more data that is infiltrating lots of different fields that weren’t particularly ready for that,” Professor Armbrust said. “It’s all a little overwhelming.”</p>
<p>The Human Microbiome Project, which is sequencing the microbial populations in the human digestive tract, has generated about a million times as much sequence data as a single human genome, said C. Titus Brown, a bioinformatics specialist at Michigan State University.</p>
<p>“It’s not at all clear what you do with that data,” he said. “Doing a comprehensive analysis of it is essentially impossible at the moment.”</p>
<p>Other scientific fields, like particle physics and astronomy, handle huge amounts of data. In those fields, however, much of the data is generated by a few huge accelerators or observatories, said Eugene Kolker, chief data officer at Seattle Children’s Hospital.</p>
<p>“In the life sciences, anyone can produce so much data, and it’s happening in thousands of different labs throughout the world,” he said.</p>
<p>Moreover, DNA is just part of the story. To truly understand biology, researchers are gathering data on the RNA, proteins and chemicals in cells. That data can be even more voluminous than data on genes. And those different types of data have to be integrated.</p>
<p>“We have these giant piles of data and no way to connect them” said H. Steven Wiley, a biologist at the Pacific Northwest National Laboratory. He added, “I’m sitting in front of a pile of data that we’ve been trying to analyze for the last year and a half.”</p>
<p>Still, many say the situation will be manageable. Jay Flatley, chief executive of Illumina, the leading supplier of sequencing machines, said he did not think information handling was a bottleneck or that it was causing people to hold off on buying new sequencers.</p>
<p>Researchers are increasingly turning to cloud computing so they do not have to buy so many of their own computers and disk drives.</p>
<p>Google might help as well.</p>
<p>“Google has enough capacity to do all of genomics in a day,” said Dr. Schatz of Cold Spring Harbor, who is trying to apply Google’s techniques to genomics data. Prodded by Senator Charles E. Schumer, Democrat of New York, Google is exploring cooperation with Cold Spring Harbor.</p>
<p>Google’s venture capital arm recently invested in DNAnexus, a bioinformatics company. DNAnexus and Google plan to host their own copy of the federal sequence archive that had once looked as if it might be closed.</p>
<p>The amount of data stored for a human genome will drop sharply. Sequencers produce huge amounts of raw data that then has to be analyzed and processed by software to produce the result.</p>
<p>With the field still young, many researchers store all the raw data, so it can be re-analyzed if better software is developed in the future.</p>
<p>In uncertain times, “scientists cling to their data,” said David J. Dooling, assistant director of the genome institute at Washington University in St. Louis.</p>
<p>But there is now so much raw data that it is becoming not feasible to re-analyze it. So researchers will increasingly store just the final results. In the case of human genomes, they might store even less — only the difference between a particular genome and some reference genome.</p>
<p>Professor Brown of Michigan State said: “We are going to have to come up with really clever ways to throw away data so we can see new stuff.”</p>
<p>This article has been revised to reflect the following correction:</p>
<p>Correction: December 2, 2011</p>
<p>A picture caption on Thursday with an article about the struggle to store, transmit and analyze the growing volume of data from gene sequencing misstated what W. Richard McCombie, the researcher pictured, was examining. It is a glass device known as a flow cell, to which the DNA from cells is attached; it does not contain cells themselves.</p>
  <div class="forum-topic-navigation clear-block">
              <a href="/~hrangwal/node/326" class="topic-next" title="Go to next forum topic">Project Presentation Signups ›</a>
      </div>
  </div>
      <div class="meta">
      Tags: <ul class="links inline"><li class="taxonomy_term_8 first last"><a href="/~hrangwal/forum/8" rel="tag" title="This forum is for discussion of class related activities.">CS 750 Discussion Forum</a></li>
</ul>          <div class="links"><ul class="links inline"><li class="comment_forbidden first"><span><a href="/~hrangwal/user/login?destination=comment/reply/367%2523comment-form">Login</a> to post comments</span></li>
<li class="print_mail last"><a href="/~hrangwal/printmail/367" title="Send this page by e-mail." class="print-mail" rel="nofollow">Send to friend</a></li>
</ul></div>
          </div>
  </div> <!-- node -->
				</div><!-- /content -->
											</div><!-- /content-area -->
			<hr class="break"/>
			<div id="bottom-blocks" class="container-16">
				<hr class="rule-bottom grid-16"/>
				<div id="bottom-left" class="grid-4">
									</div><!-- /bottom-left -->
				<div id="bottom-center-left" class="grid-4">
									</div><!-- /bottom-center-left -->
				<div id="bottom-center-right" class="grid-4">
									</div><!-- /bottom-center-right -->
				<div id="bottom-right" class="grid-4">
					<div id="block-system-0" class="block block-system block-bottom_right">
    <a href="http://drupal.org"><img src="/~hrangwal/misc/powered-blue-80x15.png" alt="Powered by Drupal, an open source content management system" title="Powered by Drupal, an open source content management system" width="80" height="15" /></a></div><!-- /block-system-0 -->
<hr class="break"/>				</div><!-- /bottom-right -->
			</div><!-- /bottom-blocks -->
			<hr class="break"/>
			<div id="footer-area" class="container-16">
				<hr class="rule-top grid-16"/>
				<div id="footer" class="grid-16">
					(c) Rangwala 2008, George Mason University, Fairfax, VA				</div><!-- /footer -->
			</div><!-- /footer-area -->
			<hr class="break"/>
		</div><!-- /page-wrapper -->
			</body>
</html>
